<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.550">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>quarto5 - Actinobacteriota and WREF</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">quarto5</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./about.html" aria-current="page"> 
<span class="menu-text">Project</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#abstract" id="toc-abstract" class="nav-link active" data-scroll-target="#abstract">Abstract</a></li>
  <li><a href="#motivating-reasons" id="toc-motivating-reasons" class="nav-link" data-scroll-target="#motivating-reasons">Motivating Reasons</a></li>
  <li><a href="#introduction" id="toc-introduction" class="nav-link" data-scroll-target="#introduction">Introduction</a>
  <ul class="collapse">
  <li><a href="#data-collection-site-wref" id="toc-data-collection-site-wref" class="nav-link" data-scroll-target="#data-collection-site-wref">Data Collection Site: WREF</a></li>
  <li><a href="#bacterial-phylum-acidobacteriota" id="toc-bacterial-phylum-acidobacteriota" class="nav-link" data-scroll-target="#bacterial-phylum-acidobacteriota">Bacterial Phylum: Acidobacteriota</a></li>
  </ul></li>
  <li><a href="#methods" id="toc-methods" class="nav-link" data-scroll-target="#methods">Methods</a></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results">Results</a>
  <ul class="collapse">
  <li><a href="#wref-site" id="toc-wref-site" class="nav-link" data-scroll-target="#wref-site">WREF Site</a></li>
  <li><a href="#acidobacteriota" id="toc-acidobacteriota" class="nav-link" data-scroll-target="#acidobacteriota">Acidobacteriota</a></li>
  </ul></li>
  <li><a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion">Discussion</a></li>
  <li><a href="#conclusions" id="toc-conclusions" class="nav-link" data-scroll-target="#conclusions">Conclusions</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Actinobacteriota and WREF</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="abstract" class="level1">
<h1>Abstract</h1>
<p>Utilizing the coding knowledge and skills learned in Biology 476: Evolutionary Genomics and Bioinformatics, students built a github repository and Quatro website. The website includes graphs and tables created by the students that showcase data from NEON metagenomes. The results are visual representations of data from the WREF collection site in Vancouver, WA as well as data on the bacteria phylum acidobacteriota. RESULTS</p>
</section>
<section id="motivating-reasons" class="level1">
<h1>Motivating Reasons</h1>
<p>The motivating reasons behind this project were to further develop my skills in R Posit Cloud, to practice integration and manipulation of data, and to learn about acidobacteriota and the WREF site. I took Biology 478: Human Genome Analysis in the fall of 2023 and this gave me some foundational skills for working with data in R. Through my engagement with this project, I have learned more about data wrangling, ggplot, ggtrees, quatro website building, and github repositories. These will be very useful to my future career in biological research as I will be able to analyze and present data that I have collected myself for presentations and papers. The topic of this project, acidobacteriota and WREF, have given me insight into a particular bacterial phylum and its evolutionary lineage. It also shed light onto how data collection sites are chosen, managed, and utilized.</p>
</section>
<section id="introduction" class="level1">
<h1>Introduction</h1>
<section id="data-collection-site-wref" class="level2">
<h2 class="anchored" data-anchor-id="data-collection-site-wref">Data Collection Site: WREF</h2>
<p>The Wind River Experimental Forest is a field site about 40 miles NE of Vancouver, WA in the Gifford Pinchot National Forest. This is in Neon’s Pacific Northwest Domain (D16). The area is populated by douglas firs and western hemlocks of diverse ages. The region’s climate is temperate with an average temperature of 9.2°C and average precipitation of 87.6 in. In the valley of volcanoes, Mt. Rainier and Mt. St.&nbsp;Helens, the soil here is comprised of volcanic ashe, pumice, basalt bedrock, and andesite (leftover from lava flows). Among the animals living in this area are elk, bobcats, cougars, black bears, various birds, and many small mammals. After being used as a seedling nursery for post-wildfire forest rehabilitation for almost a century, the NEON site was established in 2006 and sampling for the project began in 2018.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="WREFsitepretty.jpeg" class="img-fluid quarto-figure quarto-figure-center figure-img"></p>
<figcaption>A beautiful view of a lake in the WREF site</figcaption>
</figure>
</div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="WREFmap.png" class="img-fluid quarto-figure quarto-figure-center figure-img"></p>
<figcaption>Map of the WREF site</figcaption>
</figure>
</div>
</section>
<section id="bacterial-phylum-acidobacteriota" class="level2">
<h2 class="anchored" data-anchor-id="bacterial-phylum-acidobacteriota">Bacterial Phylum: Acidobacteriota</h2>
<p>This phylum of bacteria is gram-negative and are most prevalent in soils (though they are present in a myriad of different mediums) (@Wikipedia). The name “acidobacteriota” comes from their acidophilic– or acid loving– nature. Many species thrive in acidic environments. The majority of them are also aerobic while a minority in anaerobic. The first species of this phylum was characterized in the early 90’s (Acidobacterium capsulatum), however, the phylum itself was not identified until the 2010’s (@huber_editorial_2022. Within this phylum 22 genera and 40 species have been identified.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="Acidobacterium.jpg" class="img-fluid quarto-figure quarto-figure-center figure-img"></p>
<figcaption>Acidobacterium cf.&nbsp;capsulatum</figcaption>
</figure>
</div>
</section>
</section>
<section id="methods" class="level1">
<h1>Methods</h1>
</section>
<section id="results" class="level1">
<h1>Results</h1>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.1.4     ✔ readr     2.1.5
✔ forcats   1.0.0     ✔ stringr   1.5.1
✔ ggplot2   3.5.1     ✔ tibble    3.2.1
✔ lubridate 1.9.3     ✔ tidyr     1.3.1
✔ purrr     1.0.2     
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
ℹ Use the conflicted package (&lt;http://conflicted.r-lib.org/&gt;) to force all conflicts to become errors</code></pre>
</div>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(knitr)</span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(DT)</span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(plotly)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'plotly'

The following object is masked from 'package:ggplot2':

    last_plot

The following object is masked from 'package:stats':

    filter

The following object is masked from 'package:graphics':

    layout</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs <span class="ot">&lt;-</span> <span class="fu">read_csv</span>(<span class="st">"data1/data/NEON/GOLD_Study_ID_Gs0161344_NEON.csv"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 1754 Columns: 19
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr   (8): Bin ID, Genome Name, Bin Quality, Bin Lineage, GTDB-Tk Taxonomy L...
dbl  (10): IMG Genome ID, Bin Completeness, Bin Contamination, Total Number ...
date  (1): Date Added

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_Ind <span class="ot">&lt;-</span> NEON_MAGs <span class="sc">%&gt;%</span> </span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="st">`</span><span class="at">Genome Name</span><span class="st">`</span> <span class="sc">!=</span> <span class="st">"NEON combined assembly"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>  NEON_MAGs_Ind <span class="sc">%&gt;%</span> </span>
<span id="cb8-3"><a href="#cb8-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(<span class="st">`</span><span class="at">Bin Quality</span><span class="st">`</span> <span class="sc">==</span> <span class="st">"HQ"</span>)</span>
<span id="cb8-4"><a href="#cb8-4" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-219ddffe46abb038fbc5" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-219ddffe46abb038fbc5">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99"],["3300060645_49","3300060645_54","3300060645_61","3300060647_18","3300060648_12","3300060648_15","3300060650_79","3300060650_80","3300060650_84","3300060652_14","3300060652_21","3300060652_32","3300060653_43","3300060653_44","3300060654_21","3300060655_19","3300060655_21","3300060655_3","3300060665_13","3300060665_17","3300060665_23","3300060665_27","3300060666_11","3300060666_23","3300060668_6","3300060670_25","3300060670_50","3300060671_28","3300060695_94","3300060745_26","3300060746_23","3300060747_32","3300060780_4","3300060780_44","3300060883_22","3300060883_51","3300060883_56","3300060884_26","3300060885_22","3300060885_57","3300060889_15","3300060895_11","3300060895_20","3300060895_27","3300060895_45","3300060897_27","3300060914_17","3300060914_9","3300060916_104","3300060916_96","3300060917_27","3300061175_29","3300061175_30","3300061175_37","3300061179_23","3300061179_66","3300061202_14","3300061205_33","3300061211_12","3300061211_16","3300061211_17","3300061211_43","3300061211_51","3300061212_3","3300061213_22","3300061213_35","3300061213_41","3300061213_49","3300061213_55","3300061213_8","3300061213_89","3300061216_50","3300061216_65","3300061216_8","3300061535_63","3300061536_79","3300061536_8","3300061537_28","3300061537_50","3300061537_61","3300061537_65","3300061537_67","3300061537_69","3300061537_70","3300061537_75","3300061537_89","3300061625_2","3300061637_13","3300061637_39","3300061637_41","3300061638_42","3300061638_63","3300061638_66","3300061639_18","3300061639_24","3300061639_48","3300061639_66","3300061641_139","3300061641_92"],["Terrestrial soil microbial communities from Caribou Creek Watershed, Alaska, USA - BONA_006-O-20210707-comp-1","Terrestrial soil microbial communities from Caribou Creek Watershed, Alaska, USA - BONA_006-O-20210707-comp-1","Terrestrial soil microbial communities from Caribou Creek Watershed, Alaska, USA - BONA_006-O-20210707-comp-1","Terrestrial soil microbial communities from Yellowstone NP, Wyoming, USA - YELL_009-M-20210706-comp-1","Terrestrial soil microbial communities from Yellowstone NP, Wyoming, USA - YELL_002-M-20210706-comp-1","Terrestrial soil microbial communities from Yellowstone NP, Wyoming, USA - YELL_002-M-20210706-comp-1","Terrestrial soil microbial communities from Healy, Denali National Park, Alaska, USA - HEAL_048-O-20210622-comp-1","Terrestrial soil microbial communities from Healy, Denali National Park, Alaska, USA - HEAL_048-O-20210622-comp-1","Terrestrial soil microbial communities from Healy, Denali National Park, Alaska, USA - HEAL_048-O-20210622-comp-1","Terrestrial soil microbial communities from Niwot Ridge, Colorado, USA - NIWO_005-M-20210726-comp-1","Terrestrial soil microbial communities from Niwot Ridge, Colorado, USA - NIWO_005-M-20210726-comp-1","Terrestrial soil microbial communities from Niwot Ridge, Colorado, USA - NIWO_005-M-20210726-comp-1","Terrestrial soil microbial communities from Niwot Ridge, Colorado, USA - NIWO_004-M-20210726-comp-1","Terrestrial soil microbial communities from Niwot Ridge, Colorado, USA - NIWO_004-M-20210726-comp-1","Terrestrial soil microbial communities from Niwot Ridge, Colorado, USA - NIWO_002-M-20210728-comp-1","Terrestrial soil microbial communities from Wind River Experimental Forest, Washington, USA - WREF_073-O-20210623-comp-1","Terrestrial soil microbial communities from Wind River Experimental Forest, Washington, USA - WREF_073-O-20210623-comp-1","Terrestrial soil microbial communities from Wind River Experimental Forest, Washington, USA - WREF_073-O-20210623-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_002-O-20210720-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_002-O-20210720-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_002-O-20210720-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_002-O-20210720-comp-1","Terrestrial soil microbial communities from Niwot Ridge, Colorado, USA - NIWO_001-O-20210728-comp-1","Terrestrial soil microbial communities from Niwot Ridge, Colorado, USA - NIWO_001-O-20210728-comp-1","Terrestrial soil microbial communities from Toolik Field Station, Alaska, USA - TOOL_043-O-20210803-comp-1","Terrestrial soil microbial communities from Toolik Field Station, Alaska, USA - TOOL_041-O-20210803-comp-1","Terrestrial soil microbial communities from Toolik Field Station, Alaska, USA - TOOL_041-O-20210803-comp-1","Terrestrial soil microbial communities from Toolik Field Station, Alaska, USA - TOOL_003-O-20210805-comp-1","Terrestrial soil microbial communities from Toolik Field Station, Alaska, USA - TOOL_006-O-20210804-comp-1","Terrestrial soil microbial communities from Toolik Field Station, Alaska, USA - TOOL_002-O-20210804-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_003-M-20210726-comp-1","Terrestrial soil microbial communities from Chase Lake Wetlands, North Dakota, USA - WOOD_005-M-20210708-comp-1","Terrestrial soil microbial communities from Wind River Experimental Forest, Washington, USA - WREF_003-M-20210622-comp-1","Terrestrial soil microbial communities from Wind River Experimental Forest, Washington, USA - WREF_003-M-20210622-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_005-O-20210728-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_005-O-20210728-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_005-O-20210728-comp-1","Terrestrial soil microbial communities from Chase Lake Wetlands, North Dakota, USA - WOOD_042-M-20210712-comp-1","Terrestrial soil microbial communities from Chase Lake Wetlands, North Dakota, USA - WOOD_004-M-20210714-comp-1","Terrestrial soil microbial communities from Chase Lake Wetlands, North Dakota, USA - WOOD_004-M-20210714-comp-1","Terrestrial soil microbial communities from Wind River Experimental Forest, Washington, USA - WREF_004-O-20210622-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_005-M-20210728-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_005-M-20210728-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_005-M-20210728-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_005-M-20210728-comp-1","Terrestrial soil microbial communities from Chase Lake Wetlands, North Dakota, USA - WOOD_001-M-20210714-comp-1","Terrestrial soil microbial communities from Guanica State Forest and Biosphere Reserve, Puerto Rico - GUAN_043-M-20210921-comp-1","Terrestrial soil microbial communities from Guanica State Forest and Biosphere Reserve, Puerto Rico - GUAN_043-M-20210921-comp-1","Terrestrial soil microbial communities from Konza Prairie Bio Station, Kansas, USA - KONZ_045-M-20210721-comp-1","Terrestrial soil microbial communities from Konza Prairie Bio Station, Kansas, USA - KONZ_045-M-20210721-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_004-M-20210726-comp-1","Terrestrial soil microbial communities from Yellowstone NP, Wyoming, USA - YELL_012-O-20210708-comp-1","Terrestrial soil microbial communities from Yellowstone NP, Wyoming, USA - YELL_012-O-20210708-comp-1","Terrestrial soil microbial communities from Yellowstone NP, Wyoming, USA - YELL_012-O-20210708-comp-1","Terrestrial soil microbial communities from Wind River Experimental Forest, Washington, USA - WREF_001-O-20210621-comp-1","Terrestrial soil microbial communities from Wind River Experimental Forest, Washington, USA - WREF_001-O-20210621-comp-1","Terrestrial soil microbial communities from Great Basin, Onaqui, Utah, USA - ONAQ_008-M-20210524-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_040-M-20210503-comp-1","Terrestrial soil microbial communities from Great Basin, Onaqui, Utah, USA - ONAQ_004-M-20210525-comp-1","Terrestrial soil microbial communities from Great Basin, Onaqui, Utah, USA - ONAQ_004-M-20210525-comp-1","Terrestrial soil microbial communities from Great Basin, Onaqui, Utah, USA - ONAQ_004-M-20210525-comp-1","Terrestrial soil microbial communities from Great Basin, Onaqui, Utah, USA - ONAQ_004-M-20210525-comp-1","Terrestrial soil microbial communities from Great Basin, Onaqui, Utah, USA - ONAQ_004-M-20210525-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_038-M-20210504-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_033-M-20210505-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_033-M-20210505-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_033-M-20210505-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_033-M-20210505-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_033-M-20210505-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_033-M-20210505-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_033-M-20210505-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_006-M-20210506-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_006-M-20210506-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_006-M-20210506-comp-1","Terrestrial soil microbial communities from Great Basin, Onaqui, Utah, USA - ONAQ_002-M-20210524-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_032-M-20210504-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_032-M-20210504-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_003-M-20210506-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_003-M-20210506-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_003-M-20210506-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_003-M-20210506-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_003-M-20210506-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_003-M-20210506-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_003-M-20210506-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_003-M-20210506-comp-1","Terrestrial soil microbial communities from National Grasslands LBJ, Texas, USA - CLBJ_003-M-20210506-comp-1","Terrestrial soil microbial communities from Yellowstone NP, Wyoming, USA - YELL_051-M-20210705-comp-1","Terrestrial soil microbial communities from Yellowstone NP, Wyoming, USA - YELL_003-M-20210708-comp-1","Terrestrial soil microbial communities from Yellowstone NP, Wyoming, USA - YELL_003-M-20210708-comp-1","Terrestrial soil microbial communities from Yellowstone NP, Wyoming, USA - YELL_003-M-20210708-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_004-O-20210726-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_004-O-20210726-comp-1","Terrestrial soil microbial communities from Lower Teakettle, Sierra National Forest, California, USA - TEAK_004-O-20210726-comp-1","Terrestrial soil microbial communities from Niwot Ridge, Colorado, USA - NIWO_004-O-20210726-comp-1","Terrestrial soil microbial communities from Niwot Ridge, Colorado, USA - NIWO_004-O-20210726-comp-1","Terrestrial soil microbial communities from Niwot Ridge, Colorado, USA - NIWO_004-O-20210726-comp-1","Terrestrial soil microbial communities from Niwot Ridge, Colorado, USA - NIWO_004-O-20210726-comp-1","Terrestrial soil microbial communities from Chase Lake Wetlands, North Dakota, USA - WOOD_024-O-20210714-comp-1","Terrestrial soil microbial communities from Chase Lake Wetlands, North Dakota, USA - WOOD_024-O-20210714-comp-1"],[3300060645,3300060645,3300060645,3300060647,3300060648,3300060648,3300060650,3300060650,3300060650,3300060652,3300060652,3300060652,3300060653,3300060653,3300060654,3300060655,3300060655,3300060655,3300060665,3300060665,3300060665,3300060665,3300060666,3300060666,3300060668,3300060670,3300060670,3300060671,3300060695,3300060745,3300060746,3300060747,3300060780,3300060780,3300060883,3300060883,3300060883,3300060884,3300060885,3300060885,3300060889,3300060895,3300060895,3300060895,3300060895,3300060897,3300060914,3300060914,3300060916,3300060916,3300060917,3300061175,3300061175,3300061175,3300061179,3300061179,3300061202,3300061205,3300061211,3300061211,3300061211,3300061211,3300061211,3300061212,3300061213,3300061213,3300061213,3300061213,3300061213,3300061213,3300061213,3300061216,3300061216,3300061216,3300061535,3300061536,3300061536,3300061537,3300061537,3300061537,3300061537,3300061537,3300061537,3300061537,3300061537,3300061537,3300061625,3300061637,3300061637,3300061637,3300061638,3300061638,3300061638,3300061639,3300061639,3300061639,3300061639,3300061641,3300061641],["HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ","HQ"],["Bacteria; Actinomycetota; Actinomycetes; Micrococcales; Microbacteriaceae","Bacteria; Actinomycetota; Actinomycetes; Mycobacteriales; Mycobacteriaceae; Mycobacterium","Bacteria; Actinomycetota; Actinomycetes; Mycobacteriales; Mycobacteriaceae; Mycobacterium",null,null,"Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales; Conexibacteraceae; Conexibacter; Conexibacter woesei","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria","Bacteria; Actinomycetota","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria; Pseudomonadota; Betaproteobacteria; Burkholderiales; Oxalobacteraceae; Herbaspirillum","Bacteria; Pseudomonadota; Gammaproteobacteria; Xanthomonadales; Xanthomonadaceae","Bacteria","Bacteria; Pseudomonadota; Alphaproteobacteria; Hyphomicrobiales","Bacteria","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria","Bacteria","Bacteria; Verrucomicrobiota","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales; unclassified; unclassified; Solirubrobacterales bacterium URHD0059","Bacteria; Myxococcota","Bacteria; Gemmatimonadota","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria","Bacteria","Bacteria; Pseudomonadota; Betaproteobacteria","Bacteria; Planctomycetota","Bacteria; Actinomycetota","Bacteria; Bacteroidota; Chitinophagia; Chitinophagales; Chitinophagaceae","Bacteria; Pseudomonadota; Alphaproteobacteria; Hyphomicrobiales; Hyphomicrobiaceae; Hyphomicrobium; Hyphomicrobium sp. NDB2Meth4","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria; Verrucomicrobiota","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria; Verrucomicrobiota","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria; Actinomycetota","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria","Bacteria","Bacteria; Actinomycetota; Thermoleophilia","Bacteria; Pseudomonadota; Alphaproteobacteria; Hyphomicrobiales","Bacteria","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria; Actinomycetota; Thermoleophilia",null,"Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria; Actinomycetota; Actinomycetes; Propionibacteriales","Bacteria; Pseudomonadota; Alphaproteobacteria; Hyphomicrobiales; Hyphomicrobiaceae; Methyloceanibacter",null,"Bacteria; Actinomycetota; Thermoleophilia","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales; unclassified; unclassified; Solirubrobacterales bacterium URHD0059","Bacteria; Pseudomonadota; Alphaproteobacteria; Sphingomonadales","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria; Acidobacteriota; Terriglobia; Terriglobales; Acidobacteriaceae",null,"Bacteria; Actinomycetota; Thermoleophilia","Bacteria","Bacteria; Actinomycetota; Actinomycetes","Bacteria","Bacteria; Actinomycetota; Actinomycetes","Bacteria; Actinomycetota; Thermoleophilia","Bacteria; Actinomycetota; Actinomycetes; Pseudonocardiales; Pseudonocardiaceae; Actinophytocola","Bacteria","Bacteria; Actinomycetota","Bacteria; Myxococcota; Myxococcia; Myxococcales; Anaeromyxobacteraceae; Anaeromyxobacter","Bacteria; Chlamydiota; Chlamydiia; Parachlamydiales","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria","Bacteria; Pseudomonadota; Alphaproteobacteria; Hyphomicrobiales; Hyphomicrobiaceae; Rhodoplanes; Rhodoplanes sp. Z2-YC6860","Bacteria","Bacteria","Archaea; Nitrososphaerota; Nitrososphaeria; Nitrososphaerales; Nitrososphaeraceae","Bacteria","Bacteria","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales; Conexibacteraceae; Conexibacter","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria; Pseudomonadota; Alphaproteobacteria; Hyphomicrobiales; Hyphomicrobiaceae; Methyloceanibacter",null,null,null,"Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria; Actinomycetota","Bacteria; Acidobacteriota",null,"Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales",null,null,"Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria; Pseudomonadota; Gammaproteobacteria","Bacteria; Actinomycetota; Thermoleophilia; Solirubrobacterales","Bacteria; Pseudomonadota; Alphaproteobacteria","Bacteria; Acidobacteriota; Terriglobia; Terriglobales; Acidobacteriaceae","Bacteria; Pseudomonadota","Bacteria; Actinomycetota; Actinomycetes","Bacteria; Pseudomonadota; Alphaproteobacteria",null,"Bacteria"],["Bacteria; Actinobacteriota; Actinomycetia; Actinomycetales; Microbacteriaceae; Lacisediminihabitans","Bacteria; Actinobacteriota; Actinomycetia; Mycobacteriales; Mycobacteriaceae; Mycobacterium","Bacteria; Actinobacteriota; Actinomycetia; Mycobacteriales; Mycobacteriaceae; Mycobacterium","Bacteria; Verrucomicrobiota; Verrucomicrobiae; Chthoniobacterales; UBA10450; AV80","Bacteria; Verrucomicrobiota; Verrucomicrobiae; Chthoniobacterales; UBA10450; AV80","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Baekduia","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Palsa-744","Bacteria; Eremiobacterota; Eremiobacteria; Baltobacterales; Baltobacteraceae","Bacteria; Actinobacteriota; Acidimicrobiia; Acidimicrobiales; RAAP-2; Palsa-461","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Palsa-744","Bacteria; Proteobacteria; Gammaproteobacteria; Burkholderiales; Burkholderiaceae; Herbaspirillum","Bacteria; Proteobacteria; Gammaproteobacteria; Xanthomonadales; Xanthomonadaceae; Luteimonas","Bacteria; Acidobacteriota; Acidobacteriae; Acidoferrales; UBA7541; Acidoferrum","Bacteria; Proteobacteria; Alphaproteobacteria; Rhizobiales; Xanthobacteraceae; VAZQ01","Bacteria; Acidobacteriota; Thermoanaerobaculia; UBA5704; UBA5704; UBA5704","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Palsa-744","Bacteria; Acidobacteriota; Acidobacteriae; Acidobacteriales; SbA1; Sulfotelmatobacter","Bacteria; Acidobacteriota; Thermoanaerobaculia; UBA5704; UBA5704","Bacteria; Verrucomicrobiota; Verrucomicrobiae; Chthoniobacterales; UBA10450; Palsa-1382","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Baekduia","Bacteria; Myxococcota; Polyangia; Polyangiales; Polyangiaceae","Bacteria; Gemmatimonadota; Gemmatimonadetes; Gemmatimonadales; Gemmatimonadaceae; UBA4720","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Palsa-744","Bacteria; Eremiobacterota; Eremiobacteria; Baltobacterales; Baltobacteraceae; Elarobacter","Bacteria; Myxococcota; Polyangia; Palsa-1104_A; Fen-1088","Bacteria; Proteobacteria; Gammaproteobacteria; Burkholderiales; SG8-39; SCGC-AG-212-J23","Bacteria; Planctomycetota; Planctomycetia; Planctomycetales; Planctomycetaceae","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Palsa-744; Palsa-744 sp017882375","Bacteria; Bacteroidota; Bacteroidia; Chitinophagales; Chitinophagaceae; Ginsengibacter","Bacteria; Proteobacteria; Alphaproteobacteria; Rhizobiales; Hyphomicrobiaceae; Hyphomicrobium_A","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; 70-9","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; 70-9","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; 70-9; VAYN01","Bacteria; Verrucomicrobiota; Verrucomicrobiae; Pedosphaerales; UBA11358; UBA11358","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Palsa-744","Bacteria; Verrucomicrobiota; Verrucomicrobiae; Chthoniobacterales; UBA10450; Palsa-1382","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Palsa-744","Bacteria; Actinobacteriota; UBA4738; UBA4738; UBA4738","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; 70-9","Bacteria; Eisenbacteria; RBG-16-71-46; JABDJR01","Bacteria; Acidobacteriota; Thermoanaerobaculia; UBA5704; UBA5704","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Thermoleophilaceae","Bacteria; Proteobacteria; Alphaproteobacteria; Rhizobiales; Beijerinckiaceae; JAFASC01","Bacteria; Acidobacteriota; Thermoanaerobaculia; Gp7-AA8; Gp7-AA8; QHVT01","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Palsa-744","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; 70-9","Bacteria; Acidobacteriota; Blastocatellia; Pyrinomonadales; Pyrinomonadaceae; JACMLC01","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; 70-9; VAYN01","Bacteria; Actinobacteriota; Actinomycetia; Propionibacteriales; Propionibacteriaceae","Bacteria; Proteobacteria; Alphaproteobacteria; Rhizobiales; Methyloligellaceae; Methyloceanibacter","Bacteria; Acidobacteriota; Blastocatellia; Pyrinomonadales; Pyrinomonadaceae; UBA11740","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; 70-9","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Baekduia","Bacteria; Proteobacteria; Alphaproteobacteria; Sphingomonadales; Sphingomonadaceae; Allosphingosinicella","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Palsa-744","Bacteria; Acidobacteriota; Acidobacteriae; Acidobacteriales; Acidobacteriaceae; Edaphobacter","Bacteria; Acidobacteriota; Blastocatellia; Pyrinomonadales; Pyrinomonadaceae; UBA11740","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; 70-9","Bacteria; Gemmatimonadota; Gemmatimonadetes; JACCXV01","Bacteria; Actinobacteriota; Actinomycetia; Jiangellales; Jiangellaceae","Bacteria; Gemmatimonadota; Gemmatimonadetes; Gemmatimonadales; GWC2-71-9; WVYX01","Bacteria; Actinobacteriota; Acidimicrobiia; IMCC26256; PALSA-555; JACDBD01","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; 70-9; VGBV01","Bacteria; Actinobacteriota; Actinomycetia; Mycobacteriales; Pseudonocardiaceae; Actinophytocola","Bacteria; Acidobacteriota; Vicinamibacteria; Vicinamibacterales; UBA2999; Gp6-AA45","Bacteria; Actinobacteriota; Thermoleophilia; Gaiellales; Gaiellaceae; Palsa-739","Bacteria; Myxococcota; Myxococcia; Myxococcales; Anaeromyxobacteraceae; R267","Bacteria; Chlamydiota; Chlamydiia; Chlamydiales; Rhabdochlamydiaceae; CAJCIW01","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Palsa-744","Bacteria; Acidobacteriota; Blastocatellia; Pyrinomonadales; Pyrinomonadaceae; UBA11740","Bacteria; Proteobacteria; Alphaproteobacteria; Rhizobiales; Xanthobacteraceae; Z2-YC6860","Bacteria; Acidobacteriota; Blastocatellia; UBA7656; UBA7656; JADGNW01","Bacteria; Acidobacteriota; Blastocatellia; Pyrinomonadales; Pyrinomonadaceae; PSRF01",null,"Bacteria; Gemmatimonadota; Gemmatimonadetes; Gemmatimonadales; GWC2-71-9; 20CM-2-65-7","Bacteria; Dormibacterota; Dormibacteria; Dormibacterales; Dormibacteraceae; CF-161","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Conexibacter","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Palsa-744","Bacteria; Proteobacteria; Alphaproteobacteria; Rhizobiales; Methyloligellaceae; Methyloceanibacter","Bacteria; Acidobacteriota; Blastocatellia; Pyrinomonadales; Pyrinomonadaceae; UBA11740","Bacteria; Eremiobacterota; Eremiobacteria; Baltobacterales; Baltobacteraceae",null,"Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; AC-38","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Palsa-744","Bacteria; Acidobacteriota; Acidobacteriae; Acidoferrales; UBA7541; Acidoferrum",null,"Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; 70-9","Bacteria; Verrucomicrobiota; Verrucomicrobiae; Chthoniobacterales; UBA10450; AV80","Bacteria; Acidobacteriota; Blastocatellia; Pyrinomonadales; Pyrinomonadaceae; UBA11740","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; 70-9; VAYN01","Bacteria; Proteobacteria; Gammaproteobacteria; REEB76; REEB76","Bacteria; Actinobacteriota; Thermoleophilia; Solirubrobacterales; Solirubrobacteraceae; Palsa-744","Bacteria; Proteobacteria; Alphaproteobacteria; Micropepsales; Micropepsaceae; Rhizomicrobium","Bacteria; Acidobacteriota; Acidobacteriae; Acidobacteriales; Acidobacteriaceae; Tous-C9LFEB","Bacteria; Proteobacteria; Alphaproteobacteria; Micavibrionales; GCA-2720935","Bacteria; Actinobacteriota; Acidimicrobiia; Acidimicrobiales; RAAP-2; Bog-473","Bacteria; Proteobacteria; Alphaproteobacteria; Micropepsales; Micropepsaceae; Rhizomicrobium","Bacteria; Gemmatimonadota; Gemmatimonadetes; Gemmatimonadales; GWC2-71-9; SPDF01","Bacteria; Myxococcota; Polyangia; Haliangiales; Haliangiaceae; JABFXX01"],["MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2","MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2"],["IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE","IMG_PIPELINE"],["2023-04-12","2023-04-12","2023-04-12","2023-04-12","2023-04-06","2023-04-06","2023-04-06","2023-04-06","2023-04-06","2023-04-12","2023-04-12","2023-04-12","2023-04-14","2023-04-14","2023-04-14","2023-04-14","2023-04-14","2023-04-14","2023-04-12","2023-04-12","2023-04-12","2023-04-12","2023-04-14","2023-04-14","2023-04-06","2023-04-13","2023-04-13","2023-04-14","2023-04-13","2023-04-14","2023-04-13","2023-05-08","2023-04-14","2023-04-14","2023-04-16","2023-04-16","2023-04-16","2023-04-16","2023-05-08","2023-05-08","2023-04-16","2023-05-02","2023-05-02","2023-05-02","2023-05-02","2023-05-02","2023-04-14","2023-04-14","2023-05-08","2023-05-08","2023-04-16","2023-04-16","2023-04-16","2023-04-16","2023-04-16","2023-04-16","2023-05-01","2023-04-16","2023-04-16","2023-04-16","2023-04-16","2023-04-16","2023-04-16","2023-04-16","2023-05-02","2023-05-02","2023-05-02","2023-05-02","2023-05-02","2023-05-02","2023-05-02","2023-05-02","2023-05-02","2023-05-02","2023-05-02","2023-05-12","2023-05-12","2023-05-09","2023-05-09","2023-05-09","2023-05-09","2023-05-09","2023-05-09","2023-05-09","2023-05-09","2023-05-09","2023-05-11","2023-05-11","2023-05-11","2023-05-11","2023-05-16","2023-05-16","2023-05-16","2023-05-16","2023-05-16","2023-05-16","2023-05-16","2023-05-11","2023-05-11"],[98.86,95.70999999999999,97.12,97.97,99.31999999999999,99.15000000000001,96.41,94.75,94.02,96.3,90.81999999999999,97.65000000000001,95.73,91.19,97.44,95.01000000000001,98.29000000000001,95.23,97.3,97.72,93.87,97.8,97.22,94.91,90.68000000000001,98.73999999999999,100,96.3,98.28,90.27,96.43000000000001,97.41,93.09999999999999,91.17,95.44,100,96.72,90.59999999999999,90.95,98.90000000000001,95.63,99.56999999999999,98.75,95.16,96.58,97.84,91.45,98.28,91.17,95.67,98.23999999999999,96.98,98.72,99.04000000000001,97.01000000000001,90.52,95.89,96.55,96.64,92.65000000000001,93.41,98.29000000000001,93.53,99.11,97.44,92.67,99.34999999999999,93.06999999999999,96.58,95.67,91.72,90.59999999999999,94.12,93.2,98.34999999999999,93.93000000000001,98.29000000000001,94.87,93.95,98.23999999999999,99.06999999999999,90.67,97.44,97.86,94.87,90.23999999999999,96.12,97.97,97.44,97.84,96.56,96.15000000000001,99.09,97.41,95.59999999999999,95.3,93.88,98.90000000000001,90.56],[0.93,1.89,1.87,4.05,2.7,2.56,2.56,1.85,1.42,0.85,0.2,1.38,1.76,4.95,3.47,0.93,1.71,1.71,3.38,1.88,4.68,2.85,1.71,1.85,3.25,1.9,1.69,1.57,1.18,0.8100000000000001,0.86,3.45,0.86,1.58,0.43,3.38,0.85,0.85,1.72,0,2.56,3.02,3.57,1.99,0.85,0.86,4.32,0.86,2.99,3.91,1.71,0.86,0.77,0,1.71,1.72,0,1.72,2.2,1.85,2.2,1.28,2.16,2.36,3.42,0.86,1.94,0.68,2.92,2.56,0.95,3.66,2.99,3.88,2.2,1.85,2.56,0.85,2.92,2.56,0.93,1.6,0,0,2.99,0.8,1.72,3.38,3.42,0,2.4,1.71,0.36,0,0,2.56,1.01,2.2,4.19],[3650100,6028930,6696428,4392762,4231496,5521678,3180829,3891917,2361272,3127234,1919911,3758818,5145105,7329490,6356002,2817963,6460987,6879896,3216429,4189599,9156303,4086406,3020630,4553088,6164008,4164638,8251080,3725598,4600390,2798814,2694968,2105148,2382334,3480982,3385652,3283220,2984054,2942400,2440312,3813530,6407577,5086393,4341878,5230152,2415064,2239898,4759799,2383318,4322239,2814947,4714347,2831712,4246695,3553301,3268510,4551928,4690955,2855376,3030511,5342043,4428439,4504422,2862097,12584079,6328382,2524703,6725840,1933951,2972180,4963100,5469026,5498645,4904529,3257367,3411637,5329381,3131698,2837304,3022183,4685634,4264261,2412473,2959741,3560613,7246247,1815998,2337072,4298213,5048805,2613766,2908253,3170403,4574941,3682429,1721789,4099444,4495495,2850670,7457808],[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,2,2,1,1,1,2,1,1,1,1,1,1,1,3,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],[2,1,1,2,2,1,1,1,1,1,1,1,2,1,1,1,1,1,1,1,1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,2,2,1,1,1,2,1,2,1,1,1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],[1,1,1,2,2,1,1,1,1,1,1,1,2,1,1,1,1,1,2,1,2,1,1,1,1,1,1,1,1,1,1,1,1,2,1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,2,1,1,1,1,1,1,1,1,1,1,2,1,2,1,1,1,1,1,2,1,1,1,1,1,1,2,1,1,1,1,1,1,1,2,2,2,1,2,1,2,2,1,1,1,1,1,1,1,1],[47,49,55,50,46,50,44,47,41,50,43,40,53,45,49,46,62,53,46,50,48,50,49,69,45,43,89,59,43,34,52,53,47,36,47,49,50,56,51,48,51,52,48,43,44,58,43,52,40,37,58,52,50,45,50,48,48,55,49,60,62,62,50,66,46,37,47,39,50,102,40,41,67,41,48,42,54,38,43,50,50,39,52,56,69,41,48,46,52,48,46,49,46,40,36,54,45,48,62],[3528,6100,7231,3871,3697,5473,3161,4182,2456,3062,1930,3499,4811,7399,5688,2866,5945,5274,3120,4187,7903,3896,2824,4492,5395,4297,7137,3725,4264,2878,2846,2223,2531,3657,3309,3168,2917,3146,2479,3345,5004,5031,4408,4695,2373,2312,4080,2505,4840,3158,4255,2827,4173,3636,2970,4013,4104,2901,2976,5610,4143,4617,3087,11604,6055,2907,6275,1906,3052,4534,5442,5165,4308,3783,3206,5558,3050,2765,3377,4179,4273,2522,3065,3480,6221,1779,2448,3832,4578,2652,2783,3053,4719,3096,1905,4312,4527,2751,7072],[8,214,510,32,5,52,71,293,188,28,39,169,47,339,111,78,47,67,109,63,109,134,30,77,393,27,107,76,196,148,37,49,122,255,13,10,13,129,9,61,296,11,101,187,16,17,47,18,326,149,15,23,14,15,19,76,33,34,39,302,97,47,173,202,362,231,114,27,86,28,50,267,225,307,57,494,8,33,122,25,20,124,14,25,72,42,52,17,5,22,6,5,62,19,51,296,188,12,861],["Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344","Gs0161344"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Bin ID<\/th>\n      <th>Genome Name<\/th>\n      <th>IMG Genome ID<\/th>\n      <th>Bin Quality<\/th>\n      <th>Bin Lineage<\/th>\n      <th>GTDB-Tk Taxonomy Lineage<\/th>\n      <th>Bin Methods<\/th>\n      <th>Created By<\/th>\n      <th>Date Added<\/th>\n      <th>Bin Completeness<\/th>\n      <th>Bin Contamination<\/th>\n      <th>Total Number of Bases<\/th>\n      <th>5s rRNA<\/th>\n      <th>16s rRNA<\/th>\n      <th>23s rRNA<\/th>\n      <th>tRNA Genes<\/th>\n      <th>Gene Count<\/th>\n      <th>Scaffold Count<\/th>\n      <th>GOLD Study ID<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[3,10,11,12,13,14,15,16,17,18]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Bin ID","targets":1},{"name":"Genome Name","targets":2},{"name":"IMG Genome ID","targets":3},{"name":"Bin Quality","targets":4},{"name":"Bin Lineage","targets":5},{"name":"GTDB-Tk Taxonomy Lineage","targets":6},{"name":"Bin Methods","targets":7},{"name":"Created By","targets":8},{"name":"Date Added","targets":9},{"name":"Bin Completeness","targets":10},{"name":"Bin Contamination","targets":11},{"name":"Total Number of Bases","targets":12},{"name":"5s rRNA","targets":13},{"name":"16s rRNA","targets":14},{"name":"23s rRNA","targets":15},{"name":"tRNA Genes","targets":16},{"name":"Gene Count","targets":17},{"name":"Scaffold Count","targets":18},{"name":"GOLD Study ID","targets":19}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs <span class="ot">&lt;-</span> <span class="fu">read_csv</span>(<span class="st">"data1/data/NEON/GOLD_Study_ID_Gs0161344_NEON.csv"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(<span class="sc">-</span><span class="fu">c</span>(<span class="st">`</span><span class="at">GOLD Study ID</span><span class="st">`</span>, <span class="st">`</span><span class="at">Bin Methods</span><span class="st">`</span>, <span class="st">`</span><span class="at">Created By</span><span class="st">`</span>, <span class="st">`</span><span class="at">Date Added</span><span class="st">`</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="st">"Assembly Type"</span> <span class="ot">=</span> <span class="fu">case_when</span>(<span class="st">`</span><span class="at">Genome Name</span><span class="st">`</span> <span class="sc">==</span> <span class="st">"NEON combined assembly"</span> <span class="sc">~</span> <span class="st">`</span><span class="at">Genome Name</span><span class="st">`</span>,</span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a>                            <span class="cn">TRUE</span> <span class="sc">~</span> <span class="st">"Individual"</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate_at</span>(<span class="st">"Assembly Type"</span>, str_replace, <span class="st">"NEON combined assembly"</span>, <span class="st">"Combined"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb9-6"><a href="#cb9-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">separate</span>(<span class="st">`</span><span class="at">GTDB-Tk Taxonomy Lineage</span><span class="st">`</span>, <span class="fu">c</span>(<span class="st">"Domain"</span>, <span class="st">"Phylum"</span>, <span class="st">"Class"</span>, <span class="st">"Order"</span>, <span class="st">"Family"</span>, <span class="st">"Genus"</span>), <span class="st">"; "</span>, <span class="at">remove =</span> <span class="cn">FALSE</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb9-7"><a href="#cb9-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate_at</span>(<span class="st">"Genome Name"</span>, str_replace, <span class="st">"Terrestrial soil microbial communities from "</span>, <span class="st">""</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb9-8"><a href="#cb9-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">separate</span>(<span class="st">`</span><span class="at">Genome Name</span><span class="st">`</span>, <span class="fu">c</span>(<span class="st">"Site"</span>,<span class="st">"Sample Name"</span>), <span class="st">" - "</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb9-9"><a href="#cb9-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate_at</span>(<span class="st">"Sample Name"</span>, str_replace, <span class="st">"-comp-1"</span>, <span class="st">""</span>) <span class="sc">%&gt;%</span></span>
<span id="cb9-10"><a href="#cb9-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">separate</span>(<span class="st">`</span><span class="at">Sample Name</span><span class="st">`</span>, <span class="fu">c</span>(<span class="st">"Site ID"</span>,<span class="st">"subplot.layer.date"</span>), <span class="st">"_"</span>, <span class="at">remove =</span> <span class="cn">FALSE</span>,) <span class="sc">%&gt;%</span> </span>
<span id="cb9-11"><a href="#cb9-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">separate</span>(<span class="st">`</span><span class="at">subplot.layer.date</span><span class="st">`</span>, <span class="fu">c</span>(<span class="st">"Subplot"</span>, <span class="st">"Layer"</span>, <span class="st">"Date"</span>), <span class="st">"-"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 1754 Columns: 19
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr   (8): Bin ID, Genome Name, Bin Quality, Bin Lineage, GTDB-Tk Taxonomy L...
dbl  (10): IMG Genome ID, Bin Completeness, Bin Contamination, Total Number ...
date  (1): Date Added

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Expected 6 pieces. Additional pieces discarded in 29 rows [12, 32, 66, 79, 80,
88, 96, 102, 104, 240, 334, 386, 657, 790, 846, 931, 943, 983, 1041, 1095,
...].</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Expected 6 pieces. Missing pieces filled with `NA` in 429 rows [6, 7, 42, 49,
50, 55, 60, 83, 85, 97, 100, 105, 107, 113, 114, 116, 119, 125, 129, 130, ...].</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Expected 2 pieces. Missing pieces filled with `NA` in 624 rows [1131, 1132,
1133, 1134, 1135, 1136, 1137, 1138, 1139, 1140, 1141, 1142, 1143, 1144, 1145,
1146, 1147, 1148, 1149, 1150, ...].</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_bact_ind <span class="ot">&lt;-</span> NEON_MAGs <span class="sc">%&gt;%</span> </span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(Domain <span class="sc">==</span> <span class="st">"Bacteria"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="st">`</span><span class="at">Assembly Type</span><span class="st">`</span> <span class="sc">==</span> <span class="st">"Individual"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_bact_ind <span class="sc">%&gt;%</span> </span>
<span id="cb15-2"><a href="#cb15-2" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">x =</span> Phylum)) <span class="sc">+</span></span>
<span id="cb15-3"><a href="#cb15-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>() <span class="sc">+</span></span>
<span id="cb15-4"><a href="#cb15-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_flip</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="about_files/figure-html/unnamed-chunk-7-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<section id="wref-site" class="level2">
<h2 class="anchored" data-anchor-id="wref-site">WREF Site</h2>
</section>
<section id="acidobacteriota" class="level2">
<h2 class="anchored" data-anchor-id="acidobacteriota">Acidobacteriota</h2>
</section>
</section>
<section id="discussion" class="level1">
<h1>Discussion</h1>
</section>
<section id="conclusions" class="level1">
<h1>Conclusions</h1>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>