---
title: "Code"
editor: visual
always_allow_html: yes
---
```{r}
install.packages("rlang")
```

```{r}
# Bioconductor version
if (!require("BiocManager", quietly = TRUE))
    install.packages("BiocManager")

BiocManager::install("treeio")
BiocManager::install("ggtree")
BiocManager::install("ggimage")
BiocManager::install("ggtreeExtra")
```

```{r}
library(tidyverse)
library(knitr)
library(DT)
library(plotly)
library(dplyr)
library(ggplot2)
library(zoo)
library(ggimage)
library(rphylopic)
library(treeio)
library(tidytree)
library(ape)
library(TreeTools)
library(phytools)
library(ggtree)
library(TDbook)
library(ggnewscale)
library(ggtreeExtra)
library(ggstar)
```

```{r}
NEON_MAGs <- read_csv("data/GOLD_Study_ID_Gs0161344_NEON.csv")
```

```{r}
head(NEON_MAGs)
```

```{r}
str(NEON_MAGs)
```

```{r}
NEON_MAGs <- read_csv("data/GOLD_Study_ID_Gs0161344_NEON.csv") %>% 
  select(-c(`GOLD Study ID`, `Bin Methods`, `Created By`, `Date Added`)) %>% 
  mutate("Assembly Type" = case_when(`Genome Name` == "NEON combined assembly" ~ `Genome Name`,
                            TRUE ~ "Individual")) %>% 
  mutate_at("Assembly Type", str_replace, "NEON combined assembly", "Combined") %>% 
  separate(`GTDB-Tk Taxonomy Lineage`, c("Domain", "Phylum", "Class", "Order", "Family", "Genus"), "; ", remove = FALSE) %>% 
  mutate_at("Genome Name", str_replace, "Terrestrial soil microbial communities from ", "") %>% 
  separate(`Genome Name`, c("Site","Sample Name"), " - ") %>% 
  mutate_at("Sample Name", str_replace, "-comp-1", "") %>%
  separate(`Sample Name`, c("Site ID","subplot.layer.date"), "_", remove = FALSE,) %>% 
  separate(`subplot.layer.date`, c("Subplot", "Layer", "Date"), "-") 
```


```{r}
NEON_MAGs_bact_ind <- NEON_MAGs %>% 
  filter(Domain == "Bacteria") %>% 
  filter(`Assembly Type` == "Individual") 
```

```{r}
NEON_MAGs_Ind <- NEON_MAGs %>% 
  filter(`Site` != "NEON combined assembly") 
```

```{r}
NEON_MAGs_co <- NEON_MAGs %>% 
  filter(`Assembly Type` == "Combined") 
```

```{r}
NEON_MAGs_ind <- NEON_MAGs %>% 
  filter(`Assembly Type` == "Individual") 
```
```{r}
sankey_data <- NEON_MAGs_co %>% 
  select(`GTDB-Tk Taxonomy Lineage`) %>% 
  # NAs are likely Archaea
  replace_na(list(`GTDB-Tk Taxonomy Lineage` = 'Archaea')) %>% 
  # Pavian format requires p__ etc
  separate(`GTDB-Tk Taxonomy Lineage`, c("Domain", "Phylum", "Class", "Order", "Family", "Genus", "Species"), "; ") 

```

```{r}
sankey_data[] <- t(apply(sankey_data, 1, zoo::na.locf))

# Put the data into a format that can be read by the Sankey App

sankey_data <- sankey_data %>% 
  unite(col = "classification", c(Domain, Phylum, Class, Order, Family, Genus, Species), sep='; ') %>% 
  mutate_at("classification", str_replace, "Archaea", "d__Archaea") %>% 
  mutate_at("classification", str_replace, "Bacteria", "d__Bacteria") %>%  
  mutate_at("classification", str_replace, "; ", "|p__") %>% 
  mutate_at("classification", str_replace, "; ", "|c__") %>% 
  mutate_at("classification", str_replace, "; ", "|o__") %>% 
  mutate_at("classification", str_replace, "; ", "|f__") %>% 
  mutate_at("classification", str_replace, "; ", "|g__") %>% 
  mutate_at("classification", str_replace, "; ", "|s__")  

# Create format for Pavian with counts for each taxonomic level
sankey_data_s <- sankey_data
sankey_data_g <- sankey_data
sankey_data_f <- sankey_data
sankey_data_o <- sankey_data
sankey_data_c <- sankey_data
sankey_data_p <- sankey_data
sankey_data_d <- sankey_data

sankey_data_g$classification <- sub("\\|s__.*", "", sankey_data_g$classification)  
sankey_data_f$classification <- sub("\\|g__.*", "", sankey_data_f$classification)  
sankey_data_o$classification <- sub("\\|f__.*", "", sankey_data_o$classification)  
sankey_data_c$classification <- sub("\\|o__.*", "", sankey_data_c$classification)  
sankey_data_p$classification <- sub("\\|c__.*", "", sankey_data_p$classification)  
sankey_data_d$classification <- sub("\\|p__.*", "", sankey_data_d$classification)  

sankey_data_allTaxa <- bind_rows(sankey_data_s, sankey_data_g, sankey_data_f, sankey_data_o, sankey_data_c, sankey_data_p, sankey_data_d) %>% 
  mutate(classification = as.factor(classification)) %>% 
  count(classification) %>% 
# rename for Pavian format
  rename(`#SampleID` = `classification`) %>% 
  rename(`Metaphlan2_Analysis` = `n`) 

# Write file to input to Pavian Sankey
write_tsv(sankey_data_allTaxa, "data/NEON_MAG_co_pavian.txt")
```


```{r}
NEON_MAGs_Ind_tax <- NEON_MAGs_Ind %>% 
  separate(`GTDB-Tk Taxonomy Lineage`, c("Domain", "Phylum", "Class", "Order", "Family", "Genus"), "; ", remove = FALSE) 
```

```{r}
datatable(
  NEON_MAGs_Ind_tax %>% 
    count(Phylum, sort = TRUE)
)
```

```{r}
NEON_MAGs_Ind_tax_sample <- NEON_MAGs_Ind_tax %>% 
  # Get rid of the the common string "Soil microbial communities from "
  mutate_at("Site", str_replace, "Terrestrial soil microbial communities from ", "") %>% 
  # Use the first `-` to split the column in two
  separate(`Site`, c("Site","Sample Name"), " - ") %>% 
  # Get rid of the the common string "S-comp-1"
  mutate_at("Sample Name", str_replace, "-comp-1", "") %>%
  # separate the Sample Name into Site ID and plot info
  separate(`Sample Name`, c("Site ID","subplot.layer.date"), "_", remove = FALSE,) %>% 
  # separate the plot info into 3 columns
  separate(`subplot.layer.date`, c("Subplot", "Layer", "Date"), "-") 
```

```{r}
#| label: datatable1
datatable(
  NEON_MAGs_Ind_tax_sample %>% 
    count(Site, sort = TRUE)
)
```

```{r}
#| label: datatable2
datatable(
  NEON_MAGs_Ind_tax %>% 
    count(Phylum, sort = TRUE)
)
```

```{r}
#| label: plot1
NEON_MAGs_bact_ind %>% 
  ggplot(aes(x = fct_rev(fct_infreq(Subplot)), fill = Site)) +
  geom_bar() +
  coord_flip() +
  theme(axis.text.x = element_text(angle=45, vjust=1, hjust=1), axis.text.y = element_text(size = 5)) +  
  labs(title = "MAG Counts for Each Subplot", x = "Subplot", y = "Count") +
  theme_classic()
```

```{r}
#| label: plot2
NEON_MAGs_bact_ind %>% 
ggplot(aes(x = fct_rev(fct_infreq(Site)), fill = Phylum)) +
  geom_bar() +
    coord_flip() +
   theme(axis.text.x = element_text(size=10), axis.text.y = element_text(size=5)) + 
  labs(title = "MAGs Found at Each Site (By Phylum)", x = "Site", y = "Count") 
```

```{r}
#| label: plot3
NEON_MAGs_bact_ind %>% 
   filter(is.na(Genus)) %>%
  ggplot(aes(x =fct_infreq(Phylum))) +
  geom_bar() +
  coord_flip() +
  theme(axis.text.x = element_text(angle=45, vjust=1, hjust=1), axis.text.y = element_text(size = 5)) + 
  labs(title = "Phyla with Novel Genera", x = "Phylum", y = "Count of Novel Bacteria") +
  theme_classic()
```

```{r}
WREF_Data <- NEON_MAGs %>%
  filter(`Site ID` == "WREF")
```

```{r}
#| label: plot11
WREF_Data %>% 
  ggplot(aes(x = `Bin Quality`)) +
  geom_bar() +
  theme(axis.text.x = element_text(angle=45, vjust=1, hjust=1), axis.text.y = element_text(size = 5)) + 
  labs(title = "Bacteria Quality at WREF", x = "Quality", y = "Count of Bacteria") +
  theme_classic()
```
```{r}
#| label: plot12
WREF_Data %>%
  ggplot(aes(x = `Layer`)) +
  geom_bar() +
  theme(axis.text.x = element_text(angle=45, vjust=1, hjust=1), axis.text.y = element_text(size = 5)) + 
  labs(title = "Soil Composition at WREF", x = "Layer", y = "Count of Bacteria") +
  theme_classic()
```



```{r}
#| label: plot4
WREF_Data %>% 
ggplot(aes(x = fct_infreq(Phylum), fill = Class)) +
  geom_bar() +
  coord_flip() +
  theme(axis.text.x = element_text(angle=45, vjust=1, hjust=1), axis.text.y = element_text(size = 3))+
  labs(title = "Number of MAGs Per Phylum at WREF", x = "Phylum", y = "MAGs")
```

```{r}
Acido_Data <- NEON_MAGs %>% 
  filter(Phylum == "Acidobacteriota")
```

```{r}
#| label: plot5
Acido_Data %>% 
ggplot(aes(x = fct_rev(fct_infreq(Site)), fill = Class)) +
  geom_bar(position = position_dodge2(width = 0.9, preserve = "single")) +
  coord_flip() +
   theme(axis.text.x = element_text(size=10), axis.text.y = element_text(size=5)) + 
  labs(title = "Acidobacteria Found at Each Site (By Class)", x = "Site", y = "Count") 
```

```{r}
#| label: plot6
Acido_Data %>% 
ggplot(aes(x = fct_rev(fct_infreq(Site)), fill = Order)) +
  geom_bar(position = position_dodge2(width = 0.9, preserve = "single")) +
  coord_flip() +
   theme(axis.text.x = element_text(size=10), axis.text.y = element_text(size=5)) + 
  labs(title = "Acido. Orders at Each Site", x = "Site", y = "Count") 
```

```{r}
NEON_metagenomes <- read_tsv("data/exported_img_data_Gs0161344_NEON.tsv") %>% 
  rename(`Genome Name` = `Genome Name / Sample Name`) %>% 
  filter(str_detect(`Genome Name`, 're-annotation', negate = T)) %>% 
  filter(str_detect(`Genome Name`, 'WREF plot', negate = T)) 
```

```{r}
NEON_metagenomes <- NEON_metagenomes %>% 
  # Get rid of the the common string "Soil microbial communities from "
  mutate_at("Genome Name", str_replace, "Terrestrial soil microbial communities from ", "") %>% 
  # Use the first `-` to split the column in two
  separate(`Genome Name`, c("Site","Sample Name"), " - ") %>% 
  # Get rid of the the common string "-comp-1"
  mutate_at("Sample Name", str_replace, "-comp-1", "") %>%
  # separate the Sample Name into Site ID and plot info
  separate(`Sample Name`, c("Site ID","subplot.layer.date"), "_", remove = FALSE,) %>% 
  # separate the plot info into 3 columns
  separate(`subplot.layer.date`, c("Subplot", "Layer", "Date"), "-") 
```

```{r}
NEON_chemistry <- read_tsv("data/neon_plot_soilChem1_metadata.tsv") %>% 
  # remove -COMP from genomicsSampleID
  mutate_at("genomicsSampleID", str_replace, "-COMP", "") 
```

```{r}
#| label: plot7
NEON_MAGs %>%
  left_join(NEON_metagenomes, by = "Site ID") %>%
  left_join(NEON_chemistry, by = c("Site ID" = "siteID")) %>%
  filter(Phylum == "Acidobacteriota") %>%
  ggplot(aes(x = fct_infreq(`Site ID`), y = `soilTemp`)) +
  geom_boxplot() +
  theme(axis.text.x = element_text(angle=45, vjust=1, hjust=1)) +
  labs(title = "Soil Temp by Site", x = "Site ID", y = "Soil Temp")
```

```{r}
#| label: plot8
NEON_MAGs %>%
  left_join(NEON_metagenomes, by = "Sample Name") %>%
  left_join(NEON_chemistry, by = c("Sample Name" = "genomicsSampleID")) %>%
    filter(Phylum == "Acidobacteriota") %>%
  ggplot(aes(x = fct_infreq(`Ecosystem Subtype`), y = soilTemp)) +
  geom_point(aes(color=Class)) +
  theme(axis.text.x = element_text(angle=45, vjust=1, hjust=1)) +
  labs(title = "Soil Temp by Ecosystem Subtype", x = "Ecosystem Subtype", y = "Soil Temp")
```

```{r}
#| label: plot9
NEON_MAGs %>%
  left_join(NEON_metagenomes, by = "Sample Name") %>%
  left_join(NEON_chemistry, by = c("Sample Name" = "genomicsSampleID")) %>%
    filter(Phylum == "Acidobacteriota") %>%
  ggplot(aes(x = fct_infreq(`Ecosystem Subtype`), y = soilInCaClpH)) +
  geom_point(aes(color=Class), size = 2) +
  theme(axis.text.x = element_text(angle=45, vjust=1, hjust=1)) +
  labs(title = "Soil in CaCL pH by NLCD Class", x = "Ecosystem Subtype", y = "Soil in CaCl pH") +
  theme(legend.title = element_text(size=4), legend.text=element_text(size=4))
```

```{r}
#| label: plot10
NEON_MAGs %>%
  left_join(NEON_metagenomes, by = "Sample Name") %>%
  left_join(NEON_chemistry, by = c("Sample Name" = "genomicsSampleID")) %>%
    filter(Phylum == "Acidobacteriota") %>%
  ggplot(aes(x = fct_infreq(Class), y = soilInWaterpH)) +
  geom_bin_2d() +
  theme(axis.text.x = element_text(angle=90, vjust=1, hjust=1)) +
  labs(title = "Class v Soil pH in Water", x = "Class", y = "Soil pH in Water") +
  theme(legend.title = element_text(size=4), legend.text=element_text(size=4))
```

```{r}
NEON_MAGs %>%
  left_join(NEON_metagenomes, by = "Sample Name") %>%
  left_join(NEON_chemistry, by = c("Sample Name" = "genomicsSampleID")) %>%
    filter(Phylum == "Acidobacteriota") %>%
  ggplot(aes(x = soilTemp, y = elevation)) +
  geom_point(aes(color= siteID), size = 2) +
  theme(axis.text.x = element_text(angle=90, vjust=1, hjust=1)) +
  labs(title = "Soil Temp v Elevation (By Site)", x = "Soil Temp", y = "Elevation") +
  theme(legend.title = element_text(size=4), legend.text=element_text(size=4))
```

```{r}
NEON_MAGs <- read_csv("data/GOLD_Study_ID_Gs0161344_NEON_2024_4_21.csv") %>% 
  # remove columns that are not needed for data analysis
  select(-c(`GOLD Study ID`, `Bin Methods`, `Created By`, `Date Added`, `Bin Lineage`)) %>% 
  # create a new column with the Assembly Type
  mutate("Assembly Type" = case_when(`Genome Name` == "NEON combined assembly" ~ `Genome Name`,
                            TRUE ~ "Individual")) %>% 
  mutate_at("Assembly Type", str_replace, "NEON combined assembly", "Combined") %>% 
  mutate_at("GTDB-Tk Taxonomy Lineage", str_replace, "d__", "") %>%  
  mutate_at("GTDB-Tk Taxonomy Lineage", str_replace, "p__", "") %>% 
  mutate_at("GTDB-Tk Taxonomy Lineage", str_replace, "c__", "") %>% 
  mutate_at("GTDB-Tk Taxonomy Lineage", str_replace, "o__", "") %>% 
  mutate_at("GTDB-Tk Taxonomy Lineage", str_replace, "f__", "") %>% 
  mutate_at("GTDB-Tk Taxonomy Lineage", str_replace, "g__", "") %>% 
  mutate_at("GTDB-Tk Taxonomy Lineage", str_replace, "s__", "") %>%
  separate(`GTDB-Tk Taxonomy Lineage`, c("Domain", "Phylum", "Class", "Order", "Family", "Genus", "Species"), ";", remove = FALSE) %>% 
  mutate_at("Domain", na_if,"") %>% 
  mutate_at("Phylum", na_if,"") %>% 
  mutate_at("Class", na_if,"") %>% 
  mutate_at("Order", na_if,"") %>% 
  mutate_at("Family", na_if,"") %>% 
  mutate_at("Genus", na_if,"") %>% 
  mutate_at("Species", na_if,"") %>% 
  
  # Get rid of the the common string "Soil microbial communities from "
  mutate_at("Genome Name", str_replace, "Terrestrial soil microbial communities from ", "") %>% 
  # Use the first `-` to split the column in two
  separate(`Genome Name`, c("Site","Sample Name"), " - ") %>% 
  # Get rid of the the common string "S-comp-1"
  mutate_at("Sample Name", str_replace, "-comp-1", "") %>%
  # separate the Sample Name into Site ID and plot info
  separate(`Sample Name`, c("Site ID","subplot.layer.date"), "_", remove = FALSE,) %>% 
  # separate the plot info into 3 columns
  separate(`subplot.layer.date`, c("Subplot", "Layer", "Date"), "-") 
```

```{r}
NEON_metagenomes <- read_tsv("data/exported_img_data_Gs0161344_NEON.tsv") %>% 
  select(-c(`Domain`, `Sequencing Status`, `Sequencing Center`)) %>% 
  rename(`Genome Name` = `Genome Name / Sample Name`) %>% 
  filter(str_detect(`Genome Name`, 're-annotation', negate = T)) %>% 
  filter(str_detect(`Genome Name`, 'WREF plot', negate = T)) 
```
```{r}
NEON_metagenomes <- NEON_metagenomes %>% 
  # Get rid of the the common string "Soil microbial communities from "
  mutate_at("Genome Name", str_replace, "Terrestrial soil microbial communities from ", "") %>% 
  # Use the first `-` to split the column in two
  separate(`Genome Name`, c("Site","Sample Name"), " - ") %>% 
  # Get rid of the the common string "-comp-1"
  mutate_at("Sample Name", str_replace, "-comp-1", "") %>%
  # separate the Sample Name into Site ID and plot info
  separate(`Sample Name`, c("Site ID","subplot.layer.date"), "_", remove = FALSE,) %>% 
  # separate the plot info into 3 columns
  separate(`subplot.layer.date`, c("Subplot", "Layer", "Date"), "-") 
```
```{r}
NEON_chemistry <- read_tsv("data/neon_plot_soilChem1_metadata.tsv") %>% 
  # remove -COMP from genomicsSampleID
  mutate_at("genomicsSampleID", str_replace, "-COMP", "") 
```

```{r}
NEON_MAGs_metagenomes_chemistry <- NEON_MAGs %>% 
  left_join(NEON_metagenomes, by = "Sample Name") %>% 
  left_join(NEON_chemistry, by = c("Sample Name" = "genomicsSampleID")) %>% 
  rename("label" = "Bin ID")
```

```{r}
tree_arc <- read.tree("NEON/gtdbtk.ar53.decorated.tree")
tree_bac <- read.tree("data/gtdbtk.bac120.decorated.tree")
```

```{r}
# Make a vector with the internal node lables
node_vector_bac = c(tree_bac$tip.label,tree_bac$node.label)

# Search for your Phylum or Class
grep("Acidobacteriota", node_vector_bac, value = TRUE)
```

```{r}
match(grep("Acidobacteriota", node_vector_bac, value = TRUE), node_vector_bac)
```

```{r}
tree_bac_node_Acidobacteriota <- Preorder(tree_bac)
tree_Acidobacteriota <- Subtree(tree_bac_node_Acidobacteriota, 2673)
```

```{r,fig.height=29, fig.width=14}
ggtree(tree_Acidobacteriota) +
  geom_tiplab(size=3) +
  xlim(0,28) 
```

```{r,fig.height=32, fig.width=14}
#| label: tree1
ggtree(tree_Acidobacteriota, layout="circular")  %<+%
  NEON_MAGs_metagenomes_chemistry + 
  geom_tiplab(size=2, hjust=-.1) +
  xlim(0,28) +
  geom_point(mapping=aes(color=Class, shape = `Assembly Type`)) 
```

```{r}
#| label: tree2
ggtree(tree_bac, layout="circular", branch.length="none") +
    geom_hilight(node=2673, fill="steelblue", alpha=.6) +
    geom_cladelab(node=2673, label="Acidobacteriota", align=TRUE, hjust=1.1,  
                  offset = 0, textcolor='steelblue', barcolor='steelblue')
```
```{r,fig.height=32, fig.width=14}
ggtree(tree_Acidobacteriota)  %<+%
  NEON_MAGs_metagenomes_chemistry + 
  geom_tiplab(size=2, hjust=-.1) +
  xlim(0,28) +
  geom_point(mapping=aes(color=`soilInWaterpH`)) 
```
```{r,fig.height=20, fig.width=20}
NEON_MAGs_metagenomes_chemistry_WREF <- NEON_MAGs_metagenomes_chemistry %>% 
  filter(`Site ID.x` == "WREF") %>% 
  filter(Phylum == "Acidobacteriota")

WREF_MAGs_label <- NEON_MAGs_metagenomes_chemistry_WREF$label

tree_bac_WREF_MAGs <-drop.tip(tree_bac,tree_bac$tip.label[-match(WREF_MAGs_label, tree_bac$tip.label)])


ggtree(tree_bac_WREF_MAGs, layout="circular")  %<+%
  NEON_MAGs_metagenomes_chemistry +
  geom_point(mapping=aes(color=Phylum)) +
  theme(legend.title=element_text(size=5), legend.text=element_text(size=5), legend.spacing.y = unit(0.02, "cm"))
      
```
```{r,fig.height=20, fig.width=20}
NEON_MAGs_metagenomes_chemistry_soilInCaClpH <- NEON_MAGs_metagenomes_chemistry %>% 
  filter(soilInCaClpH < 0) %>% 
  filter(Domain == "Bacteria")
```

```{r}
soilInCaClpH_MAGs_label <- NEON_MAGs_metagenomes_chemistry_soilInCaClpH
```
```{r}
tree_bac_soilInCaClpH_MAGs <-drop.tip(tree_bac,tree_bac$tip.label[-match(soilInCaClpH_MAGs_label, tree_bac$tip.label)])

```

```{r,fig.height=20, fig.width=20}
ggtree(tree_bac_soilInCaClpH_MAGs, layout="circular")  %<+%
  NEON_MAGs_metagenomes_chemistry +
  geom_point(mapping=aes(color=Phylum)) 
```
```{r,fig.height=20, fig.width=20}
#| label: tree3
NEON_MAGs_metagenomes_chemistry_soilInCaClpH <- NEON_MAGs_metagenomes_chemistry %>% 
  filter(soilInWaterpH < 5) %>% 
  filter(Domain == "Bacteria")
# Make a vector of the MAGs labels
soilInCaClpH_MAGs_label <- NEON_MAGs_metagenomes_chemistry_soilInCaClpH$label

# Use appropriate Bacteria or Archaea tree to select your site MAG labels
tree_bac_soilInCaClpH_MAGs <-drop.tip(tree_bac,tree_bac$tip.label[-match(soilInCaClpH_MAGs_label, tree_bac$tip.label)])

# Now make a tree with your MAGs of interest
ggtree(tree_bac_soilInCaClpH_MAGs, layout="circular")  %<+%
  NEON_MAGs_metagenomes_chemistry +
  geom_point(mapping=aes(color=Phylum)) 
```

```{r}
NEON_chemistry_WREF <- NEON_chemistry %>%
  filter(siteID == 'WREF') 
```

```{r}
#| label: plot13
  NEON_chemistry_WREF %>%
  ggplot(aes(x = soilInWaterpH, y= soilTemp)) +
  geom_point(aes(color= nlcdClass)) +
  theme(axis.text.x = element_text(angle=45, vjust=1, hjust=1), axis.text.y = element_text(size = 5)) + 
  labs(title = "Soil pH v Soil Temp at WREF", x = "Soil in Water pH", y = "Soil Temp") 
```
