<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
    <meta charset="utf-8">
    <meta name="generator" content="quarto-1.4.550">

    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


    <title>quarto5 - Code</title>
    <style>
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      div.columns{display: flex; gap: min(4vw, 1.5em);}
      div.column{flex: auto; overflow-x: auto;}
      div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
      ul.task-list{list-style: none;}
      ul.task-list li input[type="checkbox"] {
        width: 0.8em;
        margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
        vertical-align: middle;
      }
      /* CSS for syntax highlighting */
      pre > code.sourceCode { white-space: pre; position: relative; }
      pre > code.sourceCode > span { line-height: 1.25; }
      pre > code.sourceCode > span:empty { height: 1.2em; }
      .sourceCode { overflow: visible; }
      code.sourceCode > span { color: inherit; text-decoration: inherit; }
      div.sourceCode { margin: 1em 0; }
      pre.sourceCode { margin: 0; }
      @media screen {
      div.sourceCode { overflow: auto; }
      }
      @media print {
      pre > code.sourceCode { white-space: pre-wrap; }
      pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
      }
      pre.numberSource code
        { counter-reset: source-line 0; }
      pre.numberSource code > span
        { position: relative; left: -4em; counter-increment: source-line; }
      pre.numberSource code > span > a:first-child::before
        { content: counter(source-line);
          position: relative; left: -1em; text-align: right; vertical-align: baseline;
          border: none; display: inline-block;
          -webkit-touch-callout: none; -webkit-user-select: none;
          -khtml-user-select: none; -moz-user-select: none;
          -ms-user-select: none; user-select: none;
          padding: 0 4px; width: 4em;
        }
      pre.numberSource { margin-left: 3em;  padding-left: 4px; }
      div.sourceCode
        {   }
      @media screen {
      pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
      }
    </style>

    <style>
      body.hypothesis-enabled #quarto-embed-header {
        padding-right: 36px;
      }

      #quarto-embed-header {
        height: 3em;
        width: 100%;
        display: flex;
        justify-content: space-between;
        align-items: center;
        border-bottom: solid 1px;
      }

      #quarto-embed-header h6 {
        font-size: 1.1em;
        padding-top: 0.6em;
        margin-left: 1em;
        margin-right: 1em;
        font-weight: 400;
      }

      #quarto-embed-header a.quarto-back-link,
      #quarto-embed-header a.quarto-download-embed {
        font-size: 0.8em;
        margin-top: 1em;
        margin-bottom: 1em;
        margin-left: 1em;
        margin-right: 1em;
      }

      .quarto-back-container {
        padding-left: 0.5em;
        display: flex;
      }

      .headroom {
          will-change: transform;
          transition: transform 200ms linear;
      }

      .headroom--pinned {
          transform: translateY(0%);
      }

      .headroom--unpinned {
          transform: translateY(-100%);
      }      
    </style>

    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script><script>
    window.document.addEventListener("DOMContentLoaded", function () {

      var header = window.document.querySelector("#quarto-embed-header");
      const titleBannerEl = window.document.querySelector("body > #title-block-header");
      if (titleBannerEl) {
        titleBannerEl.style.paddingTop = header.clientHeight + "px";
      }
      const contentEl = window.document.getElementById('quarto-content');
      for (const child of contentEl.children) {
        child.style.paddingTop = header.clientHeight + "px";
        child.style.marginTop = "1em";
      }

      // Use the article root if the `back` call doesn't work. This isn't perfect
      // but should typically work
      window.quartoBackToArticle = () => {
        var currentUrl = window.location.href;
        window.history.back();
        setTimeout(() => {
            // if location was not changed in 100 ms, then there is no history back
            if(currentUrl === window.location.href){              
                // redirect to site root
                window.location.href = '/';
            }
        }, 100);
      }

      const headroom = new window.Headroom(header, {
        tolerance: 5,
        onPin: function () {
        },
        onUnpin: function () {
        },
      });
      headroom.init();
    });
    </script>

    
<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
     <script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>  <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js" integrity="sha512-c3Nl8+7g4LMSTdrm621y7kf9v3SDPnhxLNhcjFJbKECVnmZHTdo+IRO05sNLTH/D3vA6u1X32ehoLC7WFVdheg==" crossorigin="anonymous"></script>

<script type="application/javascript">define('jquery', [],function() {return window.jQuery;})</script>  
        <link rel="stylesheet" href="styles.css">
      </head>

  <body class="floating nav-fixed quarto-notebook">
    <div id="quarto-embed-header" class="headroom fixed-top bg-primary">
      
      <a onclick="window.quartoBackToArticle(); return false;" class="btn btn-primary quarto-back-link"><i class="bi bi-caret-left"></i> Back to Article</a>
      <h6><i class="bi bi-journal-code"></i> Code</h6>

            <a href="./code.qmd" class="btn btn-primary quarto-download-embed" download="code.qmd">Download Source</a>
          </div>

     <div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">quarto5</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./about.html"> 
<span class="menu-text">Project</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./code.html"> 
<span class="menu-text">Code</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#code" id="toc-code" class="nav-link active" data-scroll-target="#code">Code</a></li>
  </ul>
</nav>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">      <header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Code</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>     <div id="63ae01c2-9f72-4a2a-ab48-6bc4855a0409" class="cell markdown">
<section id="code" class="level1">
<h1>Code</h1>
</section>
</div>
<div class="cell-container"><div class="cell-decorator"><pre>In [2]:</pre></div><div id="76a0d7fc-6bfa-4e9a-ad9a-9455d5f17c9d" class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a>install.packages(<span class="st">"rlang"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Installing package into '/cloud/lib/x86_64-pc-linux-gnu-library/4.3'
(as 'lib' is unspecified)</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [3]:</pre></div><div id="a769cadd-0dae-4b02-96c5-47eab308f56e" class="cell">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Bioconductor version</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="op">!</span>require(<span class="st">"BiocManager"</span>, quietly <span class="op">=</span> TRUE))</span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a>    install.packages(<span class="st">"BiocManager"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Bioconductor version '3.18' is out-of-date; the current release version '3.19'
  is available with R version '4.4'; see https://bioconductor.org/install'getOption("repos")' replaces Bioconductor standard repositories, see
'help("repositories", package = "BiocManager")' for details.
Replacement repositories:
    CRAN: http://rspm/default/__linux__/focal/latestBioconductor version 3.18 (BiocManager 1.30.23), R 4.3.3 (2024-02-29)Warning: package(s) not installed when version(s) same as or greater than current; use
  `force = TRUE` to re-install: 'treeio'Installation paths not writeable, unable to update packages
  path: /opt/R/4.3.3/lib/R/library
  packages:
    boot, codetools, lattice, survival'getOption("repos")' replaces Bioconductor standard repositories, see
'help("repositories", package = "BiocManager")' for details.
Replacement repositories:
    CRAN: http://rspm/default/__linux__/focal/latest
Bioconductor version 3.18 (BiocManager 1.30.23), R 4.3.3 (2024-02-29)Warning: package(s) not installed when version(s) same as or greater than current; use
  `force = TRUE` to re-install: 'ggtree'Installation paths not writeable, unable to update packages
  path: /opt/R/4.3.3/lib/R/library
  packages:
    boot, codetools, lattice, survival'getOption("repos")' replaces Bioconductor standard repositories, see
'help("repositories", package = "BiocManager")' for details.
Replacement repositories:
    CRAN: http://rspm/default/__linux__/focal/latest
Bioconductor version 3.18 (BiocManager 1.30.23), R 4.3.3 (2024-02-29)Warning: package(s) not installed when version(s) same as or greater than current; use
  `force = TRUE` to re-install: 'ggimage'Installation paths not writeable, unable to update packages
  path: /opt/R/4.3.3/lib/R/library
  packages:
    boot, codetools, lattice, survival'getOption("repos")' replaces Bioconductor standard repositories, see
'help("repositories", package = "BiocManager")' for details.
Replacement repositories:
    CRAN: http://rspm/default/__linux__/focal/latest
Bioconductor version 3.18 (BiocManager 1.30.23), R 4.3.3 (2024-02-29)Warning: package(s) not installed when version(s) same as or greater than current; use
  `force = TRUE` to re-install: 'ggtreeExtra'Installation paths not writeable, unable to update packages
  path: /opt/R/4.3.3/lib/R/library
  packages:
    boot, codetools, lattice, survival</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [4]:</pre></div><div id="6192991f-ac6c-4e28-8361-d0adac322424" class="cell">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a>library(tidyverse)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.1.4     ✔ readr     2.1.5
✔ forcats   1.0.0     ✔ stringr   1.5.1
✔ ggplot2   3.5.1     ✔ tibble    3.2.1
✔ lubridate 1.9.3     ✔ tidyr     1.3.1
✔ purrr     1.0.2     
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
ℹ Use the conflicted package (&lt;http://conflicted.r-lib.org/&gt;) to force all conflicts to become errors
Attaching package: 'plotly'

The following object is masked from 'package:ggplot2':

    last_plot

The following object is masked from 'package:stats':

    filter

The following object is masked from 'package:graphics':

    layout
Attaching package: 'zoo'

The following objects are masked from 'package:base':

    as.Date, as.Date.numericYou are using rphylopic v.1.4.0. Please remember to credit PhyloPic contributors (hint: `get_attribution()`) and cite rphylopic in your work (hint: `citation("rphylopic")`).

Attaching package: 'rphylopic'

The following object is masked from 'package:ggimage':

    geom_phylopictreeio v1.26.0 For help: https://yulab-smu.top/treedata-book/

If you use the ggtree package suite in published research, please cite
the appropriate paper(s):

LG Wang, TTY Lam, S Xu, Z Dai, L Zhou, T Feng, P Guo, CW Dunn, BR
Jones, T Bradley, H Zhu, Y Guan, Y Jiang, G Yu. treeio: an R package
for phylogenetic tree input and output with richly annotated and
associated data. Molecular Biology and Evolution. 2020, 37(2):599-603.
doi: 10.1093/molbev/msz240

Guangchuang Yu. Using ggtree to visualize data on tree-like structures.
Current Protocols in Bioinformatics. 2020, 69:e96. doi:10.1002/cpbi.96

Shuangbin Xu, Lin Li, Xiao Luo, Meijun Chen, Wenli Tang, Li Zhan, Zehan
Dai, Tommy T. Lam, Yi Guan, Guangchuang Yu. Ggtree: A serialized data
object for visualization of a phylogenetic tree and annotation data.
iMeta 2022, 1(4):e56. doi:10.1002/imt2.56If you use the ggtree package suite in published research, please cite
the appropriate paper(s):

LG Wang, TTY Lam, S Xu, Z Dai, L Zhou, T Feng, P Guo, CW Dunn, BR
Jones, T Bradley, H Zhu, Y Guan, Y Jiang, G Yu. treeio: an R package
for phylogenetic tree input and output with richly annotated and
associated data. Molecular Biology and Evolution. 2020, 37(2):599-603.
doi: 10.1093/molbev/msz240

Guangchuang Yu.  Data Integration, Manipulation and Visualization of
Phylogenetic Trees (1st edition). Chapman and Hall/CRC. 2022,
doi:10.1201/9781003279242

Attaching package: 'tidytree'

The following object is masked from 'package:treeio':

    getNodeNum

The following object is masked from 'package:stats':

    filter
Attaching package: 'ape'

The following objects are masked from 'package:tidytree':

    drop.tip, keep.tip

The following object is masked from 'package:treeio':

    drop.tip

The following object is masked from 'package:dplyr':

    where
Attaching package: 'TreeTools'

The following object is masked from 'package:tidytree':

    MRCA

The following object is masked from 'package:treeio':

    MRCALoading required package: maps

Attaching package: 'maps'

The following object is masked from 'package:purrr':

    map


Attaching package: 'phytools'

The following object is masked from 'package:TreeTools':

    as.multiPhylo

The following object is masked from 'package:treeio':

    read.newickggtree v3.10.1 For help: https://yulab-smu.top/treedata-book/

If you use the ggtree package suite in published research, please cite
the appropriate paper(s):

Guangchuang Yu, David Smith, Huachen Zhu, Yi Guan, Tommy Tsan-Yuk Lam.
ggtree: an R package for visualization and annotation of phylogenetic
trees with their covariates and other associated data. Methods in
Ecology and Evolution. 2017, 8(1):28-36. doi:10.1111/2041-210X.12628

G Yu. Data Integration, Manipulation and Visualization of Phylogenetic
Trees (1st ed.). Chapman and Hall/CRC. 2022. ISBN: 9781032233574

Guangchuang Yu, Tommy Tsan-Yuk Lam, Huachen Zhu, Yi Guan. Two methods
for mapping and visualizing associated data on phylogeny using ggtree.
Molecular Biology and Evolution. 2018, 35(12):3041-3043.
doi:10.1093/molbev/msy194

Attaching package: 'ggtree'

The following object is masked from 'package:TreeTools':

    MRCA

The following object is masked from 'package:ape':

    rotate

The following object is masked from 'package:tidyr':

    expandggtreeExtra v1.12.0 For help: https://yulab-smu.top/treedata-book/

If you use the ggtree package suite in published research, please cite
the appropriate paper(s):

S Xu, Z Dai, P Guo, X Fu, S Liu, L Zhou, W Tang, T Feng, M Chen, L
Zhan, T Wu, E Hu, Y Jiang, X Bo, G Yu. ggtreeExtra: Compact
visualization of richly annotated phylogenetic data. Molecular Biology
and Evolution. 2021, 38(9):4039-4042. doi: 10.1093/molbev/msab166</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [5]:</pre></div><div id="33217881-3ff3-457f-9d6a-3514790d5e65" class="cell">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs <span class="op">&lt;-</span> read_csv(<span class="st">"data/GOLD_Study_ID_Gs0161344_NEON.csv"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 1754 Columns: 19
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr   (8): Bin ID, Genome Name, Bin Quality, Bin Lineage, GTDB-Tk Taxonomy L...
dbl  (10): IMG Genome ID, Bin Completeness, Bin Contamination, Total Number ...
date  (1): Date Added

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [6]:</pre></div><div id="48f64ec6-90f0-4d53-a437-6d4efda1528e" class="cell">
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a>head(NEON_MAGs)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 6 × 19
  `Bin ID`        `Genome Name`      `IMG Genome ID` `Bin Quality` `Bin Lineage`
  &lt;chr&gt;           &lt;chr&gt;                        &lt;dbl&gt; &lt;chr&gt;         &lt;chr&gt;        
1 3300060887_39   Terrestrial soil …      3300060887 MQ            Archaea      
2 3300060914_30   Terrestrial soil …      3300060914 MQ            Archaea      
3 3300060650_39   Terrestrial soil …      3300060650 MQ            Archaea; Eur…
4 3300067032_2117 NEON combined ass…      3300067032 MQ            Archaea; Eur…
5 3300060650_21   Terrestrial soil …      3300060650 MQ            Archaea; Eur…
6 3300060650_24   Terrestrial soil …      3300060650 MQ            Archaea; Eur…
# ℹ 14 more variables: `GTDB-Tk Taxonomy Lineage` &lt;chr&gt;, `Bin Methods` &lt;chr&gt;,
#   `Created By` &lt;chr&gt;, `Date Added` &lt;date&gt;, `Bin Completeness` &lt;dbl&gt;,
#   `Bin Contamination` &lt;dbl&gt;, `Total Number of Bases` &lt;dbl&gt;, `5s rRNA` &lt;dbl&gt;,
#   `16s rRNA` &lt;dbl&gt;, `23s rRNA` &lt;dbl&gt;, `tRNA Genes` &lt;dbl&gt;, `Gene Count` &lt;dbl&gt;,
#   `Scaffold Count` &lt;dbl&gt;, `GOLD Study ID` &lt;chr&gt;</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [7]:</pre></div><div id="33d69cb1-4d8d-4b0c-a039-c109749b3c20" class="cell">
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a><span class="bu">str</span>(NEON_MAGs)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>spc_tbl_ [1,754 × 19] (S3: spec_tbl_df/tbl_df/tbl/data.frame)
 $ Bin ID                  : chr [1:1754] "3300060887_39" "3300060914_30" "3300060650_39" "3300067032_2117" ...
 $ Genome Name             : chr [1:1754] "Terrestrial soil microbial communities from Guanica State Forest and Biosphere Reserve, Puerto Rico - GUAN_042-"| __truncated__ "Terrestrial soil microbial communities from Guanica State Forest and Biosphere Reserve, Puerto Rico - GUAN_043-"| __truncated__ "Terrestrial soil microbial communities from Healy, Denali National Park, Alaska, USA - HEAL_048-O-20210622-comp-1" "NEON combined assembly" ...
 $ IMG Genome ID           : num [1:1754] 3.3e+09 3.3e+09 3.3e+09 3.3e+09 3.3e+09 ...
 $ Bin Quality             : chr [1:1754] "MQ" "MQ" "MQ" "MQ" ...
 $ Bin Lineage             : chr [1:1754] "Archaea" "Archaea" "Archaea; Euryarchaeota; Methanobacteria; Methanobacteriales; Methanobacteriaceae; Methanobacterium; Methanobact"| __truncated__ "Archaea; Euryarchaeota; Methanobacteria; Methanobacteriales; Methanobacteriaceae; Methanobacterium; Methanobact"| __truncated__ ...
 $ GTDB-Tk Taxonomy Lineage: chr [1:1754] NA NA NA NA ...
 $ Bin Methods             : chr [1:1754] "MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2" "MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2" "MetaBAT v2:2.15, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2" "MetaBAT v2:2.17, CheckM v1.2.1, GTDB-tk v2.1.1, GTDB database release R207_v2" ...
 $ Created By              : chr [1:1754] "IMG_PIPELINE" "IMG_PIPELINE" "IMG_PIPELINE" "IMG_PIPELINE" ...
 $ Date Added              : Date[1:1754], format: "2023-04-16" "2023-04-14" ...
 $ Bin Completeness        : num [1:1754] 89.8 96.1 76 83.7 96.7 ...
 $ Bin Contamination       : num [1:1754] 2.43 1.94 0.8 0.8 2.21 1.31 0.65 0 2.91 2.91 ...
 $ Total Number of Bases   : num [1:1754] 4009591 3938282 1690610 1598271 3294725 ...
 $ 5s rRNA                 : num [1:1754] 1 1 0 0 1 0 1 1 1 1 ...
 $ 16s rRNA                : num [1:1754] 0 0 0 0 0 1 1 0 0 0 ...
 $ 23s rRNA                : num [1:1754] 0 0 0 0 0 0 0 0 0 1 ...
 $ tRNA Genes              : num [1:1754] 41 43 25 33 50 42 46 42 27 27 ...
 $ Gene Count              : num [1:1754] 4918 4828 2012 1876 3082 ...
 $ Scaffold Count          : num [1:1754] 399 375 170 75 86 332 67 177 330 346 ...
 $ GOLD Study ID           : chr [1:1754] "Gs0161344" "Gs0161344" "Gs0161344" "Gs0161344" ...
 - attr(*, "spec")=
  .. cols(
  ..   `Bin ID` = col_character(),
  ..   `Genome Name` = col_character(),
  ..   `IMG Genome ID` = col_double(),
  ..   `Bin Quality` = col_character(),
  ..   `Bin Lineage` = col_character(),
  ..   `GTDB-Tk Taxonomy Lineage` = col_character(),
  ..   `Bin Methods` = col_character(),
  ..   `Created By` = col_character(),
  ..   `Date Added` = col_date(format = ""),
  ..   `Bin Completeness` = col_double(),
  ..   `Bin Contamination` = col_double(),
  ..   `Total Number of Bases` = col_double(),
  ..   `5s rRNA` = col_double(),
  ..   `16s rRNA` = col_double(),
  ..   `23s rRNA` = col_double(),
  ..   `tRNA Genes` = col_double(),
  ..   `Gene Count` = col_double(),
  ..   `Scaffold Count` = col_double(),
  ..   `GOLD Study ID` = col_character()
  .. )
 - attr(*, "problems")=&lt;externalptr&gt; </code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [8]:</pre></div><div id="4019ae3d-750e-4082-b3b8-3c3d2299479b" class="cell">
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs <span class="op">&lt;-</span> read_csv(<span class="st">"data/GOLD_Study_ID_Gs0161344_NEON.csv"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a>  select(<span class="op">-</span>c(`GOLD Study ID`, `Bin Methods`, `Created By`, `Date Added`)) <span class="op">%&gt;%</span> </span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a>  mutate(<span class="st">"Assembly Type"</span> <span class="op">=</span> case_when(`Genome Name` <span class="op">==</span> <span class="st">"NEON combined assembly"</span> <span class="op">~</span> `Genome Name`,</span>
<span id="cb13-4"><a href="#cb13-4" aria-hidden="true" tabindex="-1"></a>                            TRUE <span class="op">~</span> <span class="st">"Individual"</span>)) <span class="op">%&gt;%</span> </span>
<span id="cb13-5"><a href="#cb13-5" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Assembly Type"</span>, str_replace, <span class="st">"NEON combined assembly"</span>, <span class="st">"Combined"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb13-6"><a href="#cb13-6" aria-hidden="true" tabindex="-1"></a>  separate(`GTDB<span class="op">-</span>Tk Taxonomy Lineage`, c(<span class="st">"Domain"</span>, <span class="st">"Phylum"</span>, <span class="st">"Class"</span>, <span class="st">"Order"</span>, <span class="st">"Family"</span>, <span class="st">"Genus"</span>), <span class="st">"; "</span>, remove <span class="op">=</span> FALSE) <span class="op">%&gt;%</span> </span>
<span id="cb13-7"><a href="#cb13-7" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Genome Name"</span>, str_replace, <span class="st">"Terrestrial soil microbial communities from "</span>, <span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb13-8"><a href="#cb13-8" aria-hidden="true" tabindex="-1"></a>  separate(`Genome Name`, c(<span class="st">"Site"</span>,<span class="st">"Sample Name"</span>), <span class="st">" - "</span>) <span class="op">%&gt;%</span> </span>
<span id="cb13-9"><a href="#cb13-9" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Sample Name"</span>, str_replace, <span class="st">"-comp-1"</span>, <span class="st">""</span>) <span class="op">%&gt;%</span></span>
<span id="cb13-10"><a href="#cb13-10" aria-hidden="true" tabindex="-1"></a>  separate(`Sample Name`, c(<span class="st">"Site ID"</span>,<span class="st">"subplot.layer.date"</span>), <span class="st">"_"</span>, remove <span class="op">=</span> FALSE,) <span class="op">%&gt;%</span> </span>
<span id="cb13-11"><a href="#cb13-11" aria-hidden="true" tabindex="-1"></a>  separate(`subplot.layer.date`, c(<span class="st">"Subplot"</span>, <span class="st">"Layer"</span>, <span class="st">"Date"</span>), <span class="st">"-"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 1754 Columns: 19
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr   (8): Bin ID, Genome Name, Bin Quality, Bin Lineage, GTDB-Tk Taxonomy L...
dbl  (10): IMG Genome ID, Bin Completeness, Bin Contamination, Total Number ...
date  (1): Date Added

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.Warning: Expected 6 pieces. Additional pieces discarded in 29 rows [54, 232, 267, 328,
389, 400, 439, 473, 529, 531, 538, 669, 936, 1016, 1118, 1240, 1323, 1330,
1477, 1484, ...].Warning: Expected 6 pieces. Missing pieces filled with `NA` in 429 rows [46, 50, 53, 55,
56, 57, 61, 62, 65, 66, 67, 71, 76, 78, 79, 81, 89, 92, 96, 99, ...].Warning: Expected 2 pieces. Missing pieces filled with `NA` in 624 rows [4, 7, 8, 236,
237, 238, 239, 240, 241, 242, 243, 244, 245, 246, 247, 248, 249, 250, 251, 252,
...].</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [9]:</pre></div><div id="3970846e-f9d0-4aa0-83a1-78c07c20d107" class="cell">
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_bact_ind <span class="op">&lt;-</span> NEON_MAGs <span class="op">%&gt;%</span> </span>
<span id="cb15-2"><a href="#cb15-2" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(Domain <span class="op">==</span> <span class="st">"Bacteria"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb15-3"><a href="#cb15-3" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(`Assembly Type` <span class="op">==</span> <span class="st">"Individual"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [10]:</pre></div><div id="180a6350-36ea-4620-aa2c-415e1764db57" class="cell">
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_Ind <span class="op">&lt;-</span> NEON_MAGs <span class="op">%&gt;%</span> </span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(`Site` <span class="op">!=</span> <span class="st">"NEON combined assembly"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [11]:</pre></div><div id="f0608b86-c542-45dd-9760-f20e8c7d9584" class="cell">
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_co <span class="op">&lt;-</span> NEON_MAGs <span class="op">%&gt;%</span> </span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(`Assembly Type` <span class="op">==</span> <span class="st">"Combined"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [12]:</pre></div><div id="4efa6c3b-49fd-492c-848a-8a527e70cbae" class="cell">
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_ind <span class="op">&lt;-</span> NEON_MAGs <span class="op">%&gt;%</span> </span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(`Assembly Type` <span class="op">==</span> <span class="st">"Individual"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [13]:</pre></div><div id="f6921996-8ca3-4bc3-8319-acf0fe701ae1" class="cell">
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a>sankey_data <span class="op">&lt;-</span> NEON_MAGs_co <span class="op">%&gt;%</span> </span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>  select(`GTDB<span class="op">-</span>Tk Taxonomy Lineage`) <span class="op">%&gt;%</span> </span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a>  <span class="co"># NAs are likely Archaea</span></span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a>  replace_na(<span class="bu">list</span>(`GTDB<span class="op">-</span>Tk Taxonomy Lineage` <span class="op">=</span> <span class="st">'Archaea'</span>)) <span class="op">%&gt;%</span> </span>
<span id="cb19-5"><a href="#cb19-5" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Pavian format requires p__ etc</span></span>
<span id="cb19-6"><a href="#cb19-6" aria-hidden="true" tabindex="-1"></a>  separate(`GTDB<span class="op">-</span>Tk Taxonomy Lineage`, c(<span class="st">"Domain"</span>, <span class="st">"Phylum"</span>, <span class="st">"Class"</span>, <span class="st">"Order"</span>, <span class="st">"Family"</span>, <span class="st">"Genus"</span>, <span class="st">"Species"</span>), <span class="st">"; "</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Expected 7 pieces. Missing pieces filled with `NA` in 616 rows [1, 2, 3, 4, 5,
6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, ...].</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [14]:</pre></div><div id="565924f3-130c-4c10-8026-1777e6322853" class="cell">
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a>sankey_data[] <span class="op">&lt;-</span> t(<span class="bu">apply</span>(sankey_data, <span class="dv">1</span>, zoo::na.locf))</span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a><span class="co"># Put the data into a format that can be read by the Sankey App</span></span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a>sankey_data <span class="op">&lt;-</span> sankey_data <span class="op">%&gt;%</span> </span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a>  unite(col <span class="op">=</span> <span class="st">"classification"</span>, c(Domain, Phylum, Class, Order, Family, Genus, Species), sep<span class="op">=</span><span class="st">'; '</span>) <span class="op">%&gt;%</span> </span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"classification"</span>, str_replace, <span class="st">"Archaea"</span>, <span class="st">"d__Archaea"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb21-8"><a href="#cb21-8" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"classification"</span>, str_replace, <span class="st">"Bacteria"</span>, <span class="st">"d__Bacteria"</span>) <span class="op">%&gt;%</span>  </span>
<span id="cb21-9"><a href="#cb21-9" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"classification"</span>, str_replace, <span class="st">"; "</span>, <span class="st">"|p__"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb21-10"><a href="#cb21-10" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"classification"</span>, str_replace, <span class="st">"; "</span>, <span class="st">"|c__"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb21-11"><a href="#cb21-11" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"classification"</span>, str_replace, <span class="st">"; "</span>, <span class="st">"|o__"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb21-12"><a href="#cb21-12" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"classification"</span>, str_replace, <span class="st">"; "</span>, <span class="st">"|f__"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb21-13"><a href="#cb21-13" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"classification"</span>, str_replace, <span class="st">"; "</span>, <span class="st">"|g__"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb21-14"><a href="#cb21-14" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"classification"</span>, str_replace, <span class="st">"; "</span>, <span class="st">"|s__"</span>)  </span>
<span id="cb21-15"><a href="#cb21-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-16"><a href="#cb21-16" aria-hidden="true" tabindex="-1"></a><span class="co"># Create format for Pavian with counts for each taxonomic level</span></span>
<span id="cb21-17"><a href="#cb21-17" aria-hidden="true" tabindex="-1"></a>sankey_data_s <span class="op">&lt;-</span> sankey_data</span>
<span id="cb21-18"><a href="#cb21-18" aria-hidden="true" tabindex="-1"></a>sankey_data_g <span class="op">&lt;-</span> sankey_data</span>
<span id="cb21-19"><a href="#cb21-19" aria-hidden="true" tabindex="-1"></a>sankey_data_f <span class="op">&lt;-</span> sankey_data</span>
<span id="cb21-20"><a href="#cb21-20" aria-hidden="true" tabindex="-1"></a>sankey_data_o <span class="op">&lt;-</span> sankey_data</span>
<span id="cb21-21"><a href="#cb21-21" aria-hidden="true" tabindex="-1"></a>sankey_data_c <span class="op">&lt;-</span> sankey_data</span>
<span id="cb21-22"><a href="#cb21-22" aria-hidden="true" tabindex="-1"></a>sankey_data_p <span class="op">&lt;-</span> sankey_data</span>
<span id="cb21-23"><a href="#cb21-23" aria-hidden="true" tabindex="-1"></a>sankey_data_d <span class="op">&lt;-</span> sankey_data</span>
<span id="cb21-24"><a href="#cb21-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-25"><a href="#cb21-25" aria-hidden="true" tabindex="-1"></a>sankey_data_g$classification <span class="op">&lt;-</span> sub(<span class="st">"</span><span class="ch">\\</span><span class="st">|s__.*"</span>, <span class="st">""</span>, sankey_data_g$classification)  </span>
<span id="cb21-26"><a href="#cb21-26" aria-hidden="true" tabindex="-1"></a>sankey_data_f$classification <span class="op">&lt;-</span> sub(<span class="st">"</span><span class="ch">\\</span><span class="st">|g__.*"</span>, <span class="st">""</span>, sankey_data_f$classification)  </span>
<span id="cb21-27"><a href="#cb21-27" aria-hidden="true" tabindex="-1"></a>sankey_data_o$classification <span class="op">&lt;-</span> sub(<span class="st">"</span><span class="ch">\\</span><span class="st">|f__.*"</span>, <span class="st">""</span>, sankey_data_o$classification)  </span>
<span id="cb21-28"><a href="#cb21-28" aria-hidden="true" tabindex="-1"></a>sankey_data_c$classification <span class="op">&lt;-</span> sub(<span class="st">"</span><span class="ch">\\</span><span class="st">|o__.*"</span>, <span class="st">""</span>, sankey_data_c$classification)  </span>
<span id="cb21-29"><a href="#cb21-29" aria-hidden="true" tabindex="-1"></a>sankey_data_p$classification <span class="op">&lt;-</span> sub(<span class="st">"</span><span class="ch">\\</span><span class="st">|c__.*"</span>, <span class="st">""</span>, sankey_data_p$classification)  </span>
<span id="cb21-30"><a href="#cb21-30" aria-hidden="true" tabindex="-1"></a>sankey_data_d$classification <span class="op">&lt;-</span> sub(<span class="st">"</span><span class="ch">\\</span><span class="st">|p__.*"</span>, <span class="st">""</span>, sankey_data_d$classification)  </span>
<span id="cb21-31"><a href="#cb21-31" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-32"><a href="#cb21-32" aria-hidden="true" tabindex="-1"></a>sankey_data_allTaxa <span class="op">&lt;-</span> bind_rows(sankey_data_s, sankey_data_g, sankey_data_f, sankey_data_o, sankey_data_c, sankey_data_p, sankey_data_d) <span class="op">%&gt;%</span> </span>
<span id="cb21-33"><a href="#cb21-33" aria-hidden="true" tabindex="-1"></a>  mutate(classification <span class="op">=</span> <span class="im">as</span>.factor(classification)) <span class="op">%&gt;%</span> </span>
<span id="cb21-34"><a href="#cb21-34" aria-hidden="true" tabindex="-1"></a>  count(classification) <span class="op">%&gt;%</span> </span>
<span id="cb21-35"><a href="#cb21-35" aria-hidden="true" tabindex="-1"></a><span class="co"># rename for Pavian format</span></span>
<span id="cb21-36"><a href="#cb21-36" aria-hidden="true" tabindex="-1"></a>  rename(`<span class="co">#SampleID` = `classification`) %&gt;% </span></span>
<span id="cb21-37"><a href="#cb21-37" aria-hidden="true" tabindex="-1"></a>  rename(`Metaphlan2_Analysis` <span class="op">=</span> `n`) </span>
<span id="cb21-38"><a href="#cb21-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-39"><a href="#cb21-39" aria-hidden="true" tabindex="-1"></a><span class="co"># Write file to input to Pavian Sankey</span></span>
<span id="cb21-40"><a href="#cb21-40" aria-hidden="true" tabindex="-1"></a>write_tsv(sankey_data_allTaxa, <span class="st">"data/NEON_MAG_co_pavian.txt"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [15]:</pre></div><div id="f19bd3d2-c9c5-4db0-bb5d-a58616eb0f60" class="cell">
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_Ind_tax <span class="op">&lt;-</span> NEON_MAGs_Ind <span class="op">%&gt;%</span> </span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a>  separate(`GTDB<span class="op">-</span>Tk Taxonomy Lineage`, c(<span class="st">"Domain"</span>, <span class="st">"Phylum"</span>, <span class="st">"Class"</span>, <span class="st">"Order"</span>, <span class="st">"Family"</span>, <span class="st">"Genus"</span>), <span class="st">"; "</span>, remove <span class="op">=</span> FALSE) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Expected 6 pieces. Additional pieces discarded in 21 rows [51, 229, 272, 283,
302, 346, 348, 355, 675, 793, 851, 858, 951, 956, 959, 972, 981, 996, 997,
1037, ...].Warning: Expected 6 pieces. Missing pieces filled with `NA` in 282 rows [43, 47, 50, 52,
53, 54, 58, 59, 62, 63, 64, 68, 73, 75, 76, 78, 86, 89, 93, 96, ...].</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [16]:</pre></div><div id="40cbfc88-737c-4483-82a9-3175f9ef5be6" class="cell">
<div class="sourceCode cell-code" id="cb24"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a>datatable(</span>
<span id="cb24-2"><a href="#cb24-2" aria-hidden="true" tabindex="-1"></a>  NEON_MAGs_Ind_tax <span class="op">%&gt;%</span> </span>
<span id="cb24-3"><a href="#cb24-3" aria-hidden="true" tabindex="-1"></a>    count(Phylum, sort <span class="op">=</span> TRUE)</span>
<span id="cb24-4"><a href="#cb24-4" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-055fd669accc3c6f1e62" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-055fd669accc3c6f1e62">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25"],["Actinobacteriota","Proteobacteria","Acidobacteriota","Verrucomicrobiota",null,"Chloroflexota","Myxococcota","Bacteroidota","Gemmatimonadota","Methylomirabilota","Planctomycetota","Dormibacterota","Eremiobacterota","Desulfobacterota_B","Desulfobacterota","Patescibacteria","Tectomicrobia","Cyanobacteria","Myxococcota_A","Armatimonadota","Chlamydiota","Eisenbacteria","Firmicutes","Krumholzibacteriota","Nitrospirota"],[418,248,181,57,38,35,29,22,16,16,16,11,11,9,5,5,3,2,2,1,1,1,1,1,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Phylum<\/th>\n      <th>n<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":2},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Phylum","targets":1},{"name":"n","targets":2}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [17]:</pre></div><div id="8e5c61e9-919b-4554-bc02-f15b3f7aa430" class="cell">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_Ind_tax_sample <span class="op">&lt;-</span> NEON_MAGs_Ind_tax <span class="op">%&gt;%</span> </span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Get rid of the the common string "Soil microbial communities from "</span></span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Site"</span>, str_replace, <span class="st">"Terrestrial soil microbial communities from "</span>, <span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Use the first `-` to split the column in two</span></span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a>  separate(`Site`, c(<span class="st">"Site"</span>,<span class="st">"Sample Name"</span>), <span class="st">" - "</span>) <span class="op">%&gt;%</span> </span>
<span id="cb25-6"><a href="#cb25-6" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Get rid of the the common string "S-comp-1"</span></span>
<span id="cb25-7"><a href="#cb25-7" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Sample Name"</span>, str_replace, <span class="st">"-comp-1"</span>, <span class="st">""</span>) <span class="op">%&gt;%</span></span>
<span id="cb25-8"><a href="#cb25-8" aria-hidden="true" tabindex="-1"></a>  <span class="co"># separate the Sample Name into Site ID and plot info</span></span>
<span id="cb25-9"><a href="#cb25-9" aria-hidden="true" tabindex="-1"></a>  separate(`Sample Name`, c(<span class="st">"Site ID"</span>,<span class="st">"subplot.layer.date"</span>), <span class="st">"_"</span>, remove <span class="op">=</span> FALSE,) <span class="op">%&gt;%</span> </span>
<span id="cb25-10"><a href="#cb25-10" aria-hidden="true" tabindex="-1"></a>  <span class="co"># separate the plot info into 3 columns</span></span>
<span id="cb25-11"><a href="#cb25-11" aria-hidden="true" tabindex="-1"></a>  separate(`subplot.layer.date`, c(<span class="st">"Subplot"</span>, <span class="st">"Layer"</span>, <span class="st">"Date"</span>), <span class="st">"-"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Expected 2 pieces. Missing pieces filled with `NA` in 1130 rows [1, 2, 3, 4, 5,
6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, ...].</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [18]:</pre></div><div id="cell-datatable1" class="cell">
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a>datatable(</span>
<span id="cb27-2"><a href="#cb27-2" aria-hidden="true" tabindex="-1"></a>  NEON_MAGs_Ind_tax_sample <span class="op">%&gt;%</span> </span>
<span id="cb27-3"><a href="#cb27-3" aria-hidden="true" tabindex="-1"></a>    count(Site, sort <span class="op">=</span> TRUE)</span>
<span id="cb27-4"><a href="#cb27-4" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div id="datatable1" class="cell-output cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-d6489053c7f3a4a40b04" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-d6489053c7f3a4a40b04">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13"],["National Grasslands LBJ, Texas, USA","Toolik Field Station, Alaska, USA","Chase Lake Wetlands, North Dakota, USA","Wind River Experimental Forest, Washington, USA","Niwot Ridge, Colorado, USA","Lower Teakettle, Sierra National Forest, California, USA","Caribou Creek Watershed, Alaska, USA","Konza Prairie Bio Station, Kansas, USA","Great Basin, Onaqui, Utah, USA","Guanica State Forest and Biosphere Reserve, Puerto Rico","Yellowstone NP, Wyoming, USA","Healy, Denali National Park, Alaska, USA","Santa Rita Experimental Range, Tucson, Arizona, USA"],[197,149,132,100,88,81,69,69,60,54,50,42,39]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Site<\/th>\n      <th>n<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":2},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Site","targets":1},{"name":"n","targets":2}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [19]:</pre></div><div id="cell-datatable2" class="cell">
<div class="sourceCode cell-code" id="cb28"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a>datatable(</span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a>  NEON_MAGs_Ind_tax <span class="op">%&gt;%</span> </span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a>    count(Phylum, sort <span class="op">=</span> TRUE)</span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div id="datatable2" class="cell-output cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-4f24d993d4d39f0adcea" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-4f24d993d4d39f0adcea">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25"],["Actinobacteriota","Proteobacteria","Acidobacteriota","Verrucomicrobiota",null,"Chloroflexota","Myxococcota","Bacteroidota","Gemmatimonadota","Methylomirabilota","Planctomycetota","Dormibacterota","Eremiobacterota","Desulfobacterota_B","Desulfobacterota","Patescibacteria","Tectomicrobia","Cyanobacteria","Myxococcota_A","Armatimonadota","Chlamydiota","Eisenbacteria","Firmicutes","Krumholzibacteriota","Nitrospirota"],[418,248,181,57,38,35,29,22,16,16,16,11,11,9,5,5,3,2,2,1,1,1,1,1,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Phylum<\/th>\n      <th>n<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":2},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Phylum","targets":1},{"name":"n","targets":2}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [20]:</pre></div><div id="cell-plot1" class="cell">
<div class="sourceCode cell-code" id="cb29"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_bact_ind <span class="op">%&gt;%</span> </span>
<span id="cb29-2"><a href="#cb29-2" aria-hidden="true" tabindex="-1"></a>  ggplot(aes(x <span class="op">=</span> fct_rev(fct_infreq(Subplot)), fill <span class="op">=</span> Site)) <span class="op">+</span></span>
<span id="cb29-3"><a href="#cb29-3" aria-hidden="true" tabindex="-1"></a>  geom_bar() <span class="op">+</span></span>
<span id="cb29-4"><a href="#cb29-4" aria-hidden="true" tabindex="-1"></a>  coord_flip() <span class="op">+</span></span>
<span id="cb29-5"><a href="#cb29-5" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">45</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>), axis.text.y <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">5</span>)) <span class="op">+</span>  </span>
<span id="cb29-6"><a href="#cb29-6" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"MAG Counts for Each Subplot"</span>, x <span class="op">=</span> <span class="st">"Subplot"</span>, y <span class="op">=</span> <span class="st">"Count"</span>) <span class="op">+</span></span>
<span id="cb29-7"><a href="#cb29-7" aria-hidden="true" tabindex="-1"></a>  theme_classic()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot1-output-1.png" id="plot1" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [21]:</pre></div><div id="cell-plot2" class="cell">
<div class="sourceCode cell-code" id="cb30"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb30-1"><a href="#cb30-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_bact_ind <span class="op">%&gt;%</span> </span>
<span id="cb30-2"><a href="#cb30-2" aria-hidden="true" tabindex="-1"></a>ggplot(aes(x <span class="op">=</span> fct_rev(fct_infreq(Site)), fill <span class="op">=</span> Phylum)) <span class="op">+</span></span>
<span id="cb30-3"><a href="#cb30-3" aria-hidden="true" tabindex="-1"></a>  geom_bar() <span class="op">+</span></span>
<span id="cb30-4"><a href="#cb30-4" aria-hidden="true" tabindex="-1"></a>    coord_flip() <span class="op">+</span></span>
<span id="cb30-5"><a href="#cb30-5" aria-hidden="true" tabindex="-1"></a>   theme(axis.text.x <span class="op">=</span> element_text(size<span class="op">=</span><span class="dv">10</span>), axis.text.y <span class="op">=</span> element_text(size<span class="op">=</span><span class="dv">5</span>)) <span class="op">+</span> </span>
<span id="cb30-6"><a href="#cb30-6" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"MAGs Found at Each Site (By Phylum)"</span>, x <span class="op">=</span> <span class="st">"Site"</span>, y <span class="op">=</span> <span class="st">"Count"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot2-output-1.png" id="plot2" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [22]:</pre></div><div id="cell-plot3" class="cell">
<div class="sourceCode cell-code" id="cb31"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_bact_ind <span class="op">%&gt;%</span> </span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a>   <span class="bu">filter</span>(<span class="kw">is</span>.na(Genus)) <span class="op">%&gt;%</span></span>
<span id="cb31-3"><a href="#cb31-3" aria-hidden="true" tabindex="-1"></a>  ggplot(aes(x <span class="op">=</span>fct_infreq(Phylum))) <span class="op">+</span></span>
<span id="cb31-4"><a href="#cb31-4" aria-hidden="true" tabindex="-1"></a>  geom_bar() <span class="op">+</span></span>
<span id="cb31-5"><a href="#cb31-5" aria-hidden="true" tabindex="-1"></a>  coord_flip() <span class="op">+</span></span>
<span id="cb31-6"><a href="#cb31-6" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">45</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>), axis.text.y <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">5</span>)) <span class="op">+</span> </span>
<span id="cb31-7"><a href="#cb31-7" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Phyla with Novel Genera"</span>, x <span class="op">=</span> <span class="st">"Phylum"</span>, y <span class="op">=</span> <span class="st">"Count of Novel Bacteria"</span>) <span class="op">+</span></span>
<span id="cb31-8"><a href="#cb31-8" aria-hidden="true" tabindex="-1"></a>  theme_classic()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot3-output-1.png" id="plot3" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [23]:</pre></div><div id="e94f8cd8-53e9-4426-864b-c236a25b6592" class="cell">
<div class="sourceCode cell-code" id="cb32"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb32-1"><a href="#cb32-1" aria-hidden="true" tabindex="-1"></a>WREF_Data <span class="op">&lt;-</span> NEON_MAGs <span class="op">%&gt;%</span></span>
<span id="cb32-2"><a href="#cb32-2" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(`Site ID` <span class="op">==</span> <span class="st">"WREF"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [24]:</pre></div><div id="cell-plot11" class="cell">
<div class="sourceCode cell-code" id="cb33"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a>WREF_Data <span class="op">%&gt;%</span> </span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a>  ggplot(aes(x <span class="op">=</span> `Bin Quality`)) <span class="op">+</span></span>
<span id="cb33-3"><a href="#cb33-3" aria-hidden="true" tabindex="-1"></a>  geom_bar() <span class="op">+</span></span>
<span id="cb33-4"><a href="#cb33-4" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">45</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>), axis.text.y <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">5</span>)) <span class="op">+</span> </span>
<span id="cb33-5"><a href="#cb33-5" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Bacteria Quality at WREF"</span>, x <span class="op">=</span> <span class="st">"Quality"</span>, y <span class="op">=</span> <span class="st">"Count of Bacteria"</span>) <span class="op">+</span></span>
<span id="cb33-6"><a href="#cb33-6" aria-hidden="true" tabindex="-1"></a>  theme_classic()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot11-output-1.png" id="plot11" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [25]:</pre></div><div id="cell-plot12" class="cell">
<div class="sourceCode cell-code" id="cb34"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb34-1"><a href="#cb34-1" aria-hidden="true" tabindex="-1"></a>WREF_Data <span class="op">%&gt;%</span></span>
<span id="cb34-2"><a href="#cb34-2" aria-hidden="true" tabindex="-1"></a>  ggplot(aes(x <span class="op">=</span> `Layer`)) <span class="op">+</span></span>
<span id="cb34-3"><a href="#cb34-3" aria-hidden="true" tabindex="-1"></a>  geom_bar() <span class="op">+</span></span>
<span id="cb34-4"><a href="#cb34-4" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">45</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>), axis.text.y <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">5</span>)) <span class="op">+</span> </span>
<span id="cb34-5"><a href="#cb34-5" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Soil Composition at WREF"</span>, x <span class="op">=</span> <span class="st">"Layer"</span>, y <span class="op">=</span> <span class="st">"Count of Bacteria"</span>) <span class="op">+</span></span>
<span id="cb34-6"><a href="#cb34-6" aria-hidden="true" tabindex="-1"></a>  theme_classic()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot12-output-1.png" id="plot12" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [26]:</pre></div><div id="cell-plot4" class="cell">
<div class="sourceCode cell-code" id="cb35"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a>WREF_Data <span class="op">%&gt;%</span> </span>
<span id="cb35-2"><a href="#cb35-2" aria-hidden="true" tabindex="-1"></a>ggplot(aes(x <span class="op">=</span> fct_infreq(Phylum), fill <span class="op">=</span> Class)) <span class="op">+</span></span>
<span id="cb35-3"><a href="#cb35-3" aria-hidden="true" tabindex="-1"></a>  geom_bar() <span class="op">+</span></span>
<span id="cb35-4"><a href="#cb35-4" aria-hidden="true" tabindex="-1"></a>  coord_flip() <span class="op">+</span></span>
<span id="cb35-5"><a href="#cb35-5" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">45</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>), axis.text.y <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">3</span>))<span class="op">+</span></span>
<span id="cb35-6"><a href="#cb35-6" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Number of MAGs Per Phylum at WREF"</span>, x <span class="op">=</span> <span class="st">"Phylum"</span>, y <span class="op">=</span> <span class="st">"MAGs"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot4-output-1.png" id="plot4" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [27]:</pre></div><div id="f14d631b-c0c9-474c-a04f-dcd48465ead8" class="cell">
<div class="sourceCode cell-code" id="cb36"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb36-1"><a href="#cb36-1" aria-hidden="true" tabindex="-1"></a>Acido_Data <span class="op">&lt;-</span> NEON_MAGs <span class="op">%&gt;%</span> </span>
<span id="cb36-2"><a href="#cb36-2" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(Phylum <span class="op">==</span> <span class="st">"Acidobacteriota"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [28]:</pre></div><div id="cell-plot5" class="cell">
<div class="sourceCode cell-code" id="cb37"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a>Acido_Data <span class="op">%&gt;%</span> </span>
<span id="cb37-2"><a href="#cb37-2" aria-hidden="true" tabindex="-1"></a>ggplot(aes(x <span class="op">=</span> fct_rev(fct_infreq(Site)), fill <span class="op">=</span> Class)) <span class="op">+</span></span>
<span id="cb37-3"><a href="#cb37-3" aria-hidden="true" tabindex="-1"></a>  geom_bar(position <span class="op">=</span> position_dodge2(width <span class="op">=</span> <span class="fl">0.9</span>, preserve <span class="op">=</span> <span class="st">"single"</span>)) <span class="op">+</span></span>
<span id="cb37-4"><a href="#cb37-4" aria-hidden="true" tabindex="-1"></a>  coord_flip() <span class="op">+</span></span>
<span id="cb37-5"><a href="#cb37-5" aria-hidden="true" tabindex="-1"></a>   theme(axis.text.x <span class="op">=</span> element_text(size<span class="op">=</span><span class="dv">10</span>), axis.text.y <span class="op">=</span> element_text(size<span class="op">=</span><span class="dv">5</span>)) <span class="op">+</span> </span>
<span id="cb37-6"><a href="#cb37-6" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Acidobacteria Found at Each Site (By Class)"</span>, x <span class="op">=</span> <span class="st">"Site"</span>, y <span class="op">=</span> <span class="st">"Count"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot5-output-1.png" id="plot5" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [29]:</pre></div><div id="cell-plot6" class="cell">
<div class="sourceCode cell-code" id="cb38"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb38-1"><a href="#cb38-1" aria-hidden="true" tabindex="-1"></a>Acido_Data <span class="op">%&gt;%</span> </span>
<span id="cb38-2"><a href="#cb38-2" aria-hidden="true" tabindex="-1"></a>ggplot(aes(x <span class="op">=</span> fct_rev(fct_infreq(Site)), fill <span class="op">=</span> Order)) <span class="op">+</span></span>
<span id="cb38-3"><a href="#cb38-3" aria-hidden="true" tabindex="-1"></a>  geom_bar(position <span class="op">=</span> position_dodge2(width <span class="op">=</span> <span class="fl">0.9</span>, preserve <span class="op">=</span> <span class="st">"single"</span>)) <span class="op">+</span></span>
<span id="cb38-4"><a href="#cb38-4" aria-hidden="true" tabindex="-1"></a>  coord_flip() <span class="op">+</span></span>
<span id="cb38-5"><a href="#cb38-5" aria-hidden="true" tabindex="-1"></a>   theme(axis.text.x <span class="op">=</span> element_text(size<span class="op">=</span><span class="dv">10</span>), axis.text.y <span class="op">=</span> element_text(size<span class="op">=</span><span class="dv">5</span>)) <span class="op">+</span> </span>
<span id="cb38-6"><a href="#cb38-6" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Acido. Orders at Each Site"</span>, x <span class="op">=</span> <span class="st">"Site"</span>, y <span class="op">=</span> <span class="st">"Count"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot6-output-1.png" id="plot6" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [30]:</pre></div><div id="ddd8083b-93b4-4bc4-a68b-d0e16cdeb170" class="cell">
<div class="sourceCode cell-code" id="cb39"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a>NEON_metagenomes <span class="op">&lt;-</span> read_tsv(<span class="st">"data/exported_img_data_Gs0161344_NEON.tsv"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb39-2"><a href="#cb39-2" aria-hidden="true" tabindex="-1"></a>  rename(`Genome Name` <span class="op">=</span> `Genome Name <span class="op">/</span> Sample Name`) <span class="op">%&gt;%</span> </span>
<span id="cb39-3"><a href="#cb39-3" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(str_detect(`Genome Name`, <span class="st">'re-annotation'</span>, negate <span class="op">=</span> T)) <span class="op">%&gt;%</span> </span>
<span id="cb39-4"><a href="#cb39-4" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(str_detect(`Genome Name`, <span class="st">'WREF plot'</span>, negate <span class="op">=</span> T)) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 176 Columns: 46
── Column specification ────────────────────────────────────────────────────────
Delimiter: "\t"
chr (18): Domain, Sequencing Status, Study Name, Genome Name / Sample Name, ...
dbl (16): taxon_oid, IMG Genome ID, Depth In Meters, Elevation In Meters, Ge...
lgl (12): Altitude In Meters, Chlorophyll Concentration, Longhurst Code, Lon...

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [31]:</pre></div><div id="d1e6c226-ff33-4b2f-bba0-48a9a55bc940" class="cell">
<div class="sourceCode cell-code" id="cb41"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb41-1"><a href="#cb41-1" aria-hidden="true" tabindex="-1"></a>NEON_metagenomes <span class="op">&lt;-</span> NEON_metagenomes <span class="op">%&gt;%</span> </span>
<span id="cb41-2"><a href="#cb41-2" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Get rid of the the common string "Soil microbial communities from "</span></span>
<span id="cb41-3"><a href="#cb41-3" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Genome Name"</span>, str_replace, <span class="st">"Terrestrial soil microbial communities from "</span>, <span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb41-4"><a href="#cb41-4" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Use the first `-` to split the column in two</span></span>
<span id="cb41-5"><a href="#cb41-5" aria-hidden="true" tabindex="-1"></a>  separate(`Genome Name`, c(<span class="st">"Site"</span>,<span class="st">"Sample Name"</span>), <span class="st">" - "</span>) <span class="op">%&gt;%</span> </span>
<span id="cb41-6"><a href="#cb41-6" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Get rid of the the common string "-comp-1"</span></span>
<span id="cb41-7"><a href="#cb41-7" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Sample Name"</span>, str_replace, <span class="st">"-comp-1"</span>, <span class="st">""</span>) <span class="op">%&gt;%</span></span>
<span id="cb41-8"><a href="#cb41-8" aria-hidden="true" tabindex="-1"></a>  <span class="co"># separate the Sample Name into Site ID and plot info</span></span>
<span id="cb41-9"><a href="#cb41-9" aria-hidden="true" tabindex="-1"></a>  separate(`Sample Name`, c(<span class="st">"Site ID"</span>,<span class="st">"subplot.layer.date"</span>), <span class="st">"_"</span>, remove <span class="op">=</span> FALSE,) <span class="op">%&gt;%</span> </span>
<span id="cb41-10"><a href="#cb41-10" aria-hidden="true" tabindex="-1"></a>  <span class="co"># separate the plot info into 3 columns</span></span>
<span id="cb41-11"><a href="#cb41-11" aria-hidden="true" tabindex="-1"></a>  separate(`subplot.layer.date`, c(<span class="st">"Subplot"</span>, <span class="st">"Layer"</span>, <span class="st">"Date"</span>), <span class="st">"-"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Expected 2 pieces. Missing pieces filled with `NA` in 1 rows [53].</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [32]:</pre></div><div id="3f6fd4f5-d825-4a0a-8a54-ca3cdd5932be" class="cell">
<div class="sourceCode cell-code" id="cb43"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb43-1"><a href="#cb43-1" aria-hidden="true" tabindex="-1"></a>NEON_chemistry <span class="op">&lt;-</span> read_tsv(<span class="st">"data/neon_plot_soilChem1_metadata.tsv"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb43-2"><a href="#cb43-2" aria-hidden="true" tabindex="-1"></a>  <span class="co"># remove -COMP from genomicsSampleID</span></span>
<span id="cb43-3"><a href="#cb43-3" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"genomicsSampleID"</span>, str_replace, <span class="st">"-COMP"</span>, <span class="st">""</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 87 Columns: 17
── Column specification ────────────────────────────────────────────────────────
Delimiter: "\t"
chr   (5): genomicsSampleID, siteID, plotID, nlcdClass, horizon
dbl  (11): decimalLatitude, decimalLongitude, elevation, soilTemp, d15N, org...
date  (1): collectionDate

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [33]:</pre></div><div id="cell-plot7" class="cell">
<div class="sourceCode cell-code" id="cb45"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb45-1"><a href="#cb45-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs <span class="op">%&gt;%</span></span>
<span id="cb45-2"><a href="#cb45-2" aria-hidden="true" tabindex="-1"></a>  left_join(NEON_metagenomes, by <span class="op">=</span> <span class="st">"Site ID"</span>) <span class="op">%&gt;%</span></span>
<span id="cb45-3"><a href="#cb45-3" aria-hidden="true" tabindex="-1"></a>  left_join(NEON_chemistry, by <span class="op">=</span> c(<span class="st">"Site ID"</span> <span class="op">=</span> <span class="st">"siteID"</span>)) <span class="op">%&gt;%</span></span>
<span id="cb45-4"><a href="#cb45-4" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(Phylum <span class="op">==</span> <span class="st">"Acidobacteriota"</span>) <span class="op">%&gt;%</span></span>
<span id="cb45-5"><a href="#cb45-5" aria-hidden="true" tabindex="-1"></a>  ggplot(aes(x <span class="op">=</span> fct_infreq(`Site ID`), y <span class="op">=</span> `soilTemp`)) <span class="op">+</span></span>
<span id="cb45-6"><a href="#cb45-6" aria-hidden="true" tabindex="-1"></a>  geom_boxplot() <span class="op">+</span></span>
<span id="cb45-7"><a href="#cb45-7" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">45</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>)) <span class="op">+</span></span>
<span id="cb45-8"><a href="#cb45-8" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Soil Temp by Site"</span>, x <span class="op">=</span> <span class="st">"Site ID"</span>, y <span class="op">=</span> <span class="st">"Soil Temp"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in left_join(., NEON_metagenomes, by = "Site ID"): Detected an unexpected many-to-many relationship between `x` and `y`.
ℹ Row 1 of `x` matches multiple rows in `y`.
ℹ Row 12 of `y` matches multiple rows in `x`.
ℹ If a many-to-many relationship is expected, set `relationship =
  "many-to-many"` to silence this warning.Warning in left_join(., NEON_chemistry, by = c(`Site ID` = "siteID")): Detected an unexpected many-to-many relationship between `x` and `y`.
ℹ Row 1 of `x` matches multiple rows in `y`.
ℹ Row 1 of `y` matches multiple rows in `x`.
ℹ If a many-to-many relationship is expected, set `relationship =
  "many-to-many"` to silence this warning.Warning: Removed 188 rows containing non-finite outside the scale range
(`stat_boxplot()`).</code></pre>
</div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot7-output-2.png" id="plot7" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [34]:</pre></div><div id="cell-plot8" class="cell">
<div class="sourceCode cell-code" id="cb47"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb47-1"><a href="#cb47-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs <span class="op">%&gt;%</span></span>
<span id="cb47-2"><a href="#cb47-2" aria-hidden="true" tabindex="-1"></a>  left_join(NEON_metagenomes, by <span class="op">=</span> <span class="st">"Sample Name"</span>) <span class="op">%&gt;%</span></span>
<span id="cb47-3"><a href="#cb47-3" aria-hidden="true" tabindex="-1"></a>  left_join(NEON_chemistry, by <span class="op">=</span> c(<span class="st">"Sample Name"</span> <span class="op">=</span> <span class="st">"genomicsSampleID"</span>)) <span class="op">%&gt;%</span></span>
<span id="cb47-4"><a href="#cb47-4" aria-hidden="true" tabindex="-1"></a>    <span class="bu">filter</span>(Phylum <span class="op">==</span> <span class="st">"Acidobacteriota"</span>) <span class="op">%&gt;%</span></span>
<span id="cb47-5"><a href="#cb47-5" aria-hidden="true" tabindex="-1"></a>  ggplot(aes(x <span class="op">=</span> fct_infreq(`Ecosystem Subtype`), y <span class="op">=</span> soilTemp)) <span class="op">+</span></span>
<span id="cb47-6"><a href="#cb47-6" aria-hidden="true" tabindex="-1"></a>  geom_point(aes(color<span class="op">=</span>Class)) <span class="op">+</span></span>
<span id="cb47-7"><a href="#cb47-7" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">45</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>)) <span class="op">+</span></span>
<span id="cb47-8"><a href="#cb47-8" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Soil Temp by Ecosystem Subtype"</span>, x <span class="op">=</span> <span class="st">"Ecosystem Subtype"</span>, y <span class="op">=</span> <span class="st">"Soil Temp"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Removed 125 rows containing missing values or values outside the scale range
(`geom_point()`).</code></pre>
</div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot8-output-2.png" id="plot8" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [35]:</pre></div><div id="cell-plot9" class="cell">
<div class="sourceCode cell-code" id="cb49"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb49-1"><a href="#cb49-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs <span class="op">%&gt;%</span></span>
<span id="cb49-2"><a href="#cb49-2" aria-hidden="true" tabindex="-1"></a>  left_join(NEON_metagenomes, by <span class="op">=</span> <span class="st">"Sample Name"</span>) <span class="op">%&gt;%</span></span>
<span id="cb49-3"><a href="#cb49-3" aria-hidden="true" tabindex="-1"></a>  left_join(NEON_chemistry, by <span class="op">=</span> c(<span class="st">"Sample Name"</span> <span class="op">=</span> <span class="st">"genomicsSampleID"</span>)) <span class="op">%&gt;%</span></span>
<span id="cb49-4"><a href="#cb49-4" aria-hidden="true" tabindex="-1"></a>    <span class="bu">filter</span>(Phylum <span class="op">==</span> <span class="st">"Acidobacteriota"</span>) <span class="op">%&gt;%</span></span>
<span id="cb49-5"><a href="#cb49-5" aria-hidden="true" tabindex="-1"></a>  ggplot(aes(x <span class="op">=</span> fct_infreq(`Ecosystem Subtype`), y <span class="op">=</span> soilInCaClpH)) <span class="op">+</span></span>
<span id="cb49-6"><a href="#cb49-6" aria-hidden="true" tabindex="-1"></a>  geom_point(aes(color<span class="op">=</span>Class), size <span class="op">=</span> <span class="dv">2</span>) <span class="op">+</span></span>
<span id="cb49-7"><a href="#cb49-7" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">45</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>)) <span class="op">+</span></span>
<span id="cb49-8"><a href="#cb49-8" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Soil in CaCL pH by NLCD Class"</span>, x <span class="op">=</span> <span class="st">"Ecosystem Subtype"</span>, y <span class="op">=</span> <span class="st">"Soil in CaCl pH"</span>) <span class="op">+</span></span>
<span id="cb49-9"><a href="#cb49-9" aria-hidden="true" tabindex="-1"></a>  theme(legend.title <span class="op">=</span> element_text(size<span class="op">=</span><span class="dv">4</span>), legend.text<span class="op">=</span>element_text(size<span class="op">=</span><span class="dv">4</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Removed 125 rows containing missing values or values outside the scale range
(`geom_point()`).</code></pre>
</div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot9-output-2.png" id="plot9" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [36]:</pre></div><div id="cell-plot10" class="cell">
<div class="sourceCode cell-code" id="cb51"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb51-1"><a href="#cb51-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs <span class="op">%&gt;%</span></span>
<span id="cb51-2"><a href="#cb51-2" aria-hidden="true" tabindex="-1"></a>  left_join(NEON_metagenomes, by <span class="op">=</span> <span class="st">"Sample Name"</span>) <span class="op">%&gt;%</span></span>
<span id="cb51-3"><a href="#cb51-3" aria-hidden="true" tabindex="-1"></a>  left_join(NEON_chemistry, by <span class="op">=</span> c(<span class="st">"Sample Name"</span> <span class="op">=</span> <span class="st">"genomicsSampleID"</span>)) <span class="op">%&gt;%</span></span>
<span id="cb51-4"><a href="#cb51-4" aria-hidden="true" tabindex="-1"></a>    <span class="bu">filter</span>(Phylum <span class="op">==</span> <span class="st">"Acidobacteriota"</span>) <span class="op">%&gt;%</span></span>
<span id="cb51-5"><a href="#cb51-5" aria-hidden="true" tabindex="-1"></a>  ggplot(aes(x <span class="op">=</span> fct_infreq(Class), y <span class="op">=</span> soilInWaterpH)) <span class="op">+</span></span>
<span id="cb51-6"><a href="#cb51-6" aria-hidden="true" tabindex="-1"></a>  geom_bin_2d() <span class="op">+</span></span>
<span id="cb51-7"><a href="#cb51-7" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">90</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>)) <span class="op">+</span></span>
<span id="cb51-8"><a href="#cb51-8" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Class v Soil pH in Water"</span>, x <span class="op">=</span> <span class="st">"Class"</span>, y <span class="op">=</span> <span class="st">"Soil pH in Water"</span>) <span class="op">+</span></span>
<span id="cb51-9"><a href="#cb51-9" aria-hidden="true" tabindex="-1"></a>  theme(legend.title <span class="op">=</span> element_text(size<span class="op">=</span><span class="dv">4</span>), legend.text<span class="op">=</span>element_text(size<span class="op">=</span><span class="dv">4</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Removed 125 rows containing non-finite outside the scale range
(`stat_bin2d()`).</code></pre>
</div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot10-output-2.png" id="plot10" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [37]:</pre></div><div id="addb658f-fd08-4cfd-a605-54046e54bdde" class="cell">
<div class="sourceCode cell-code" id="cb53"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb53-1"><a href="#cb53-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs <span class="op">%&gt;%</span></span>
<span id="cb53-2"><a href="#cb53-2" aria-hidden="true" tabindex="-1"></a>  left_join(NEON_metagenomes, by <span class="op">=</span> <span class="st">"Sample Name"</span>) <span class="op">%&gt;%</span></span>
<span id="cb53-3"><a href="#cb53-3" aria-hidden="true" tabindex="-1"></a>  left_join(NEON_chemistry, by <span class="op">=</span> c(<span class="st">"Sample Name"</span> <span class="op">=</span> <span class="st">"genomicsSampleID"</span>)) <span class="op">%&gt;%</span></span>
<span id="cb53-4"><a href="#cb53-4" aria-hidden="true" tabindex="-1"></a>    <span class="bu">filter</span>(Phylum <span class="op">==</span> <span class="st">"Acidobacteriota"</span>) <span class="op">%&gt;%</span></span>
<span id="cb53-5"><a href="#cb53-5" aria-hidden="true" tabindex="-1"></a>  ggplot(aes(x <span class="op">=</span> soilTemp, y <span class="op">=</span> elevation)) <span class="op">+</span></span>
<span id="cb53-6"><a href="#cb53-6" aria-hidden="true" tabindex="-1"></a>  geom_point(aes(color<span class="op">=</span> siteID), size <span class="op">=</span> <span class="dv">2</span>) <span class="op">+</span></span>
<span id="cb53-7"><a href="#cb53-7" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">90</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>)) <span class="op">+</span></span>
<span id="cb53-8"><a href="#cb53-8" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Soil Temp v Elevation (By Site)"</span>, x <span class="op">=</span> <span class="st">"Soil Temp"</span>, y <span class="op">=</span> <span class="st">"Elevation"</span>) <span class="op">+</span></span>
<span id="cb53-9"><a href="#cb53-9" aria-hidden="true" tabindex="-1"></a>  theme(legend.title <span class="op">=</span> element_text(size<span class="op">=</span><span class="dv">4</span>), legend.text<span class="op">=</span>element_text(size<span class="op">=</span><span class="dv">4</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Removed 125 rows containing missing values or values outside the scale range
(`geom_point()`).</code></pre>
</div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/cell-37-output-2.png" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [38]:</pre></div><div id="4bb94f69-1e12-4438-93d2-f1df3062d74f" class="cell">
<div class="sourceCode cell-code" id="cb55"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb55-1"><a href="#cb55-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs <span class="op">&lt;-</span> read_csv(<span class="st">"data/GOLD_Study_ID_Gs0161344_NEON_2024_4_21.csv"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-2"><a href="#cb55-2" aria-hidden="true" tabindex="-1"></a>  <span class="co"># remove columns that are not needed for data analysis</span></span>
<span id="cb55-3"><a href="#cb55-3" aria-hidden="true" tabindex="-1"></a>  select(<span class="op">-</span>c(`GOLD Study ID`, `Bin Methods`, `Created By`, `Date Added`, `Bin Lineage`)) <span class="op">%&gt;%</span> </span>
<span id="cb55-4"><a href="#cb55-4" aria-hidden="true" tabindex="-1"></a>  <span class="co"># create a new column with the Assembly Type</span></span>
<span id="cb55-5"><a href="#cb55-5" aria-hidden="true" tabindex="-1"></a>  mutate(<span class="st">"Assembly Type"</span> <span class="op">=</span> case_when(`Genome Name` <span class="op">==</span> <span class="st">"NEON combined assembly"</span> <span class="op">~</span> `Genome Name`,</span>
<span id="cb55-6"><a href="#cb55-6" aria-hidden="true" tabindex="-1"></a>                            TRUE <span class="op">~</span> <span class="st">"Individual"</span>)) <span class="op">%&gt;%</span> </span>
<span id="cb55-7"><a href="#cb55-7" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Assembly Type"</span>, str_replace, <span class="st">"NEON combined assembly"</span>, <span class="st">"Combined"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-8"><a href="#cb55-8" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"GTDB-Tk Taxonomy Lineage"</span>, str_replace, <span class="st">"d__"</span>, <span class="st">""</span>) <span class="op">%&gt;%</span>  </span>
<span id="cb55-9"><a href="#cb55-9" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"GTDB-Tk Taxonomy Lineage"</span>, str_replace, <span class="st">"p__"</span>, <span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-10"><a href="#cb55-10" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"GTDB-Tk Taxonomy Lineage"</span>, str_replace, <span class="st">"c__"</span>, <span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-11"><a href="#cb55-11" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"GTDB-Tk Taxonomy Lineage"</span>, str_replace, <span class="st">"o__"</span>, <span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-12"><a href="#cb55-12" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"GTDB-Tk Taxonomy Lineage"</span>, str_replace, <span class="st">"f__"</span>, <span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-13"><a href="#cb55-13" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"GTDB-Tk Taxonomy Lineage"</span>, str_replace, <span class="st">"g__"</span>, <span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-14"><a href="#cb55-14" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"GTDB-Tk Taxonomy Lineage"</span>, str_replace, <span class="st">"s__"</span>, <span class="st">""</span>) <span class="op">%&gt;%</span></span>
<span id="cb55-15"><a href="#cb55-15" aria-hidden="true" tabindex="-1"></a>  separate(`GTDB<span class="op">-</span>Tk Taxonomy Lineage`, c(<span class="st">"Domain"</span>, <span class="st">"Phylum"</span>, <span class="st">"Class"</span>, <span class="st">"Order"</span>, <span class="st">"Family"</span>, <span class="st">"Genus"</span>, <span class="st">"Species"</span>), <span class="st">";"</span>, remove <span class="op">=</span> FALSE) <span class="op">%&gt;%</span> </span>
<span id="cb55-16"><a href="#cb55-16" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Domain"</span>, na_if,<span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-17"><a href="#cb55-17" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Phylum"</span>, na_if,<span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-18"><a href="#cb55-18" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Class"</span>, na_if,<span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-19"><a href="#cb55-19" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Order"</span>, na_if,<span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-20"><a href="#cb55-20" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Family"</span>, na_if,<span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-21"><a href="#cb55-21" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Genus"</span>, na_if,<span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-22"><a href="#cb55-22" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Species"</span>, na_if,<span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-23"><a href="#cb55-23" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb55-24"><a href="#cb55-24" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Get rid of the the common string "Soil microbial communities from "</span></span>
<span id="cb55-25"><a href="#cb55-25" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Genome Name"</span>, str_replace, <span class="st">"Terrestrial soil microbial communities from "</span>, <span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-26"><a href="#cb55-26" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Use the first `-` to split the column in two</span></span>
<span id="cb55-27"><a href="#cb55-27" aria-hidden="true" tabindex="-1"></a>  separate(`Genome Name`, c(<span class="st">"Site"</span>,<span class="st">"Sample Name"</span>), <span class="st">" - "</span>) <span class="op">%&gt;%</span> </span>
<span id="cb55-28"><a href="#cb55-28" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Get rid of the the common string "S-comp-1"</span></span>
<span id="cb55-29"><a href="#cb55-29" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Sample Name"</span>, str_replace, <span class="st">"-comp-1"</span>, <span class="st">""</span>) <span class="op">%&gt;%</span></span>
<span id="cb55-30"><a href="#cb55-30" aria-hidden="true" tabindex="-1"></a>  <span class="co"># separate the Sample Name into Site ID and plot info</span></span>
<span id="cb55-31"><a href="#cb55-31" aria-hidden="true" tabindex="-1"></a>  separate(`Sample Name`, c(<span class="st">"Site ID"</span>,<span class="st">"subplot.layer.date"</span>), <span class="st">"_"</span>, remove <span class="op">=</span> FALSE,) <span class="op">%&gt;%</span> </span>
<span id="cb55-32"><a href="#cb55-32" aria-hidden="true" tabindex="-1"></a>  <span class="co"># separate the plot info into 3 columns</span></span>
<span id="cb55-33"><a href="#cb55-33" aria-hidden="true" tabindex="-1"></a>  separate(`subplot.layer.date`, c(<span class="st">"Subplot"</span>, <span class="st">"Layer"</span>, <span class="st">"Date"</span>), <span class="st">"-"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 1754 Columns: 19
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr   (8): Bin ID, Genome Name, Bin Quality, Bin Lineage, GTDB-Tk Taxonomy L...
dbl  (10): IMG Genome ID, Bin Completeness, Bin Contamination, Total Number ...
date  (1): Date Added

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.Warning: Expected 2 pieces. Missing pieces filled with `NA` in 624 rows [1131, 1132,
1133, 1134, 1135, 1136, 1137, 1138, 1139, 1140, 1141, 1142, 1143, 1144, 1145,
1146, 1147, 1148, 1149, 1150, ...].</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [39]:</pre></div><div id="000acde0-48f4-4245-8219-b73e254b97c2" class="cell">
<div class="sourceCode cell-code" id="cb57"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb57-1"><a href="#cb57-1" aria-hidden="true" tabindex="-1"></a>NEON_metagenomes <span class="op">&lt;-</span> read_tsv(<span class="st">"data/exported_img_data_Gs0161344_NEON.tsv"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb57-2"><a href="#cb57-2" aria-hidden="true" tabindex="-1"></a>  select(<span class="op">-</span>c(`Domain`, `Sequencing Status`, `Sequencing Center`)) <span class="op">%&gt;%</span> </span>
<span id="cb57-3"><a href="#cb57-3" aria-hidden="true" tabindex="-1"></a>  rename(`Genome Name` <span class="op">=</span> `Genome Name <span class="op">/</span> Sample Name`) <span class="op">%&gt;%</span> </span>
<span id="cb57-4"><a href="#cb57-4" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(str_detect(`Genome Name`, <span class="st">'re-annotation'</span>, negate <span class="op">=</span> T)) <span class="op">%&gt;%</span> </span>
<span id="cb57-5"><a href="#cb57-5" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(str_detect(`Genome Name`, <span class="st">'WREF plot'</span>, negate <span class="op">=</span> T)) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 176 Columns: 46
── Column specification ────────────────────────────────────────────────────────
Delimiter: "\t"
chr (18): Domain, Sequencing Status, Study Name, Genome Name / Sample Name, ...
dbl (16): taxon_oid, IMG Genome ID, Depth In Meters, Elevation In Meters, Ge...
lgl (12): Altitude In Meters, Chlorophyll Concentration, Longhurst Code, Lon...

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [40]:</pre></div><div id="c7fcf873-3a95-40c9-b28f-9a5b6a67884b" class="cell">
<div class="sourceCode cell-code" id="cb59"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb59-1"><a href="#cb59-1" aria-hidden="true" tabindex="-1"></a>NEON_metagenomes <span class="op">&lt;-</span> NEON_metagenomes <span class="op">%&gt;%</span> </span>
<span id="cb59-2"><a href="#cb59-2" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Get rid of the the common string "Soil microbial communities from "</span></span>
<span id="cb59-3"><a href="#cb59-3" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Genome Name"</span>, str_replace, <span class="st">"Terrestrial soil microbial communities from "</span>, <span class="st">""</span>) <span class="op">%&gt;%</span> </span>
<span id="cb59-4"><a href="#cb59-4" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Use the first `-` to split the column in two</span></span>
<span id="cb59-5"><a href="#cb59-5" aria-hidden="true" tabindex="-1"></a>  separate(`Genome Name`, c(<span class="st">"Site"</span>,<span class="st">"Sample Name"</span>), <span class="st">" - "</span>) <span class="op">%&gt;%</span> </span>
<span id="cb59-6"><a href="#cb59-6" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Get rid of the the common string "-comp-1"</span></span>
<span id="cb59-7"><a href="#cb59-7" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"Sample Name"</span>, str_replace, <span class="st">"-comp-1"</span>, <span class="st">""</span>) <span class="op">%&gt;%</span></span>
<span id="cb59-8"><a href="#cb59-8" aria-hidden="true" tabindex="-1"></a>  <span class="co"># separate the Sample Name into Site ID and plot info</span></span>
<span id="cb59-9"><a href="#cb59-9" aria-hidden="true" tabindex="-1"></a>  separate(`Sample Name`, c(<span class="st">"Site ID"</span>,<span class="st">"subplot.layer.date"</span>), <span class="st">"_"</span>, remove <span class="op">=</span> FALSE,) <span class="op">%&gt;%</span> </span>
<span id="cb59-10"><a href="#cb59-10" aria-hidden="true" tabindex="-1"></a>  <span class="co"># separate the plot info into 3 columns</span></span>
<span id="cb59-11"><a href="#cb59-11" aria-hidden="true" tabindex="-1"></a>  separate(`subplot.layer.date`, c(<span class="st">"Subplot"</span>, <span class="st">"Layer"</span>, <span class="st">"Date"</span>), <span class="st">"-"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Expected 2 pieces. Missing pieces filled with `NA` in 1 rows [53].</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [41]:</pre></div><div id="4b376a68-7359-4cf0-b331-59e0c105270d" class="cell">
<div class="sourceCode cell-code" id="cb61"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb61-1"><a href="#cb61-1" aria-hidden="true" tabindex="-1"></a>NEON_chemistry <span class="op">&lt;-</span> read_tsv(<span class="st">"data/neon_plot_soilChem1_metadata.tsv"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb61-2"><a href="#cb61-2" aria-hidden="true" tabindex="-1"></a>  <span class="co"># remove -COMP from genomicsSampleID</span></span>
<span id="cb61-3"><a href="#cb61-3" aria-hidden="true" tabindex="-1"></a>  mutate_at(<span class="st">"genomicsSampleID"</span>, str_replace, <span class="st">"-COMP"</span>, <span class="st">""</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 87 Columns: 17
── Column specification ────────────────────────────────────────────────────────
Delimiter: "\t"
chr   (5): genomicsSampleID, siteID, plotID, nlcdClass, horizon
dbl  (11): decimalLatitude, decimalLongitude, elevation, soilTemp, d15N, org...
date  (1): collectionDate

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [42]:</pre></div><div id="09ec60de-4bf0-46a3-9bae-faa481dd122c" class="cell">
<div class="sourceCode cell-code" id="cb63"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb63-1"><a href="#cb63-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_metagenomes_chemistry <span class="op">&lt;-</span> NEON_MAGs <span class="op">%&gt;%</span> </span>
<span id="cb63-2"><a href="#cb63-2" aria-hidden="true" tabindex="-1"></a>  left_join(NEON_metagenomes, by <span class="op">=</span> <span class="st">"Sample Name"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb63-3"><a href="#cb63-3" aria-hidden="true" tabindex="-1"></a>  left_join(NEON_chemistry, by <span class="op">=</span> c(<span class="st">"Sample Name"</span> <span class="op">=</span> <span class="st">"genomicsSampleID"</span>)) <span class="op">%&gt;%</span> </span>
<span id="cb63-4"><a href="#cb63-4" aria-hidden="true" tabindex="-1"></a>  rename(<span class="st">"label"</span> <span class="op">=</span> <span class="st">"Bin ID"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [43]:</pre></div><div id="d1525243-bc2b-4e3f-82d1-fd49fc64676c" class="cell">
<div class="sourceCode cell-code" id="cb64"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb64-1"><a href="#cb64-1" aria-hidden="true" tabindex="-1"></a>tree_arc <span class="op">&lt;-</span> read.tree(<span class="st">"NEON/gtdbtk.ar53.decorated.tree"</span>)</span>
<span id="cb64-2"><a href="#cb64-2" aria-hidden="true" tabindex="-1"></a>tree_bac <span class="op">&lt;-</span> read.tree(<span class="st">"data/gtdbtk.bac120.decorated.tree"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [44]:</pre></div><div id="39938461-05ad-4773-842d-deb5e9135731" class="cell">
<div class="sourceCode cell-code" id="cb65"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb65-1"><a href="#cb65-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Make a vector with the internal node lables</span></span>
<span id="cb65-2"><a href="#cb65-2" aria-hidden="true" tabindex="-1"></a>node_vector_bac <span class="op">=</span> c(tree_bac$tip.label,tree_bac$node.label)</span>
<span id="cb65-3"><a href="#cb65-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb65-4"><a href="#cb65-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Search for your Phylum or Class</span></span>
<span id="cb65-5"><a href="#cb65-5" aria-hidden="true" tabindex="-1"></a>grep(<span class="st">"Acidobacteriota"</span>, node_vector_bac, value <span class="op">=</span> TRUE)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] "'1.0:p__Acidobacteriota'"</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [45]:</pre></div><div id="ab0455ec-e9a7-423b-a57a-4a5bfea62e1a" class="cell">
<div class="sourceCode cell-code" id="cb67"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb67-1"><a href="#cb67-1" aria-hidden="true" tabindex="-1"></a>match(grep(<span class="st">"Acidobacteriota"</span>, node_vector_bac, value <span class="op">=</span> TRUE), node_vector_bac)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 2673</code></pre>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [46]:</pre></div><div id="bbebca16-c338-4983-8053-0a05f6d5932e" class="cell">
<div class="sourceCode cell-code" id="cb69"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb69-1"><a href="#cb69-1" aria-hidden="true" tabindex="-1"></a>tree_bac_node_Acidobacteriota <span class="op">&lt;-</span> Preorder(tree_bac)</span>
<span id="cb69-2"><a href="#cb69-2" aria-hidden="true" tabindex="-1"></a>tree_Acidobacteriota <span class="op">&lt;-</span> Subtree(tree_bac_node_Acidobacteriota, <span class="dv">2673</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [47]:</pre></div><div id="7bbe9c8a-2626-45ce-9e14-5e529b12a49a" class="cell">
<div class="sourceCode cell-code" id="cb70"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb70-1"><a href="#cb70-1" aria-hidden="true" tabindex="-1"></a>ggtree(tree_Acidobacteriota) <span class="op">+</span></span>
<span id="cb70-2"><a href="#cb70-2" aria-hidden="true" tabindex="-1"></a>  geom_tiplab(size<span class="op">=</span><span class="dv">3</span>) <span class="op">+</span></span>
<span id="cb70-3"><a href="#cb70-3" aria-hidden="true" tabindex="-1"></a>  xlim(<span class="dv">0</span>,<span class="dv">28</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/cell-47-output-1.png" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [48]:</pre></div><div id="cell-tree1" class="cell">
<div class="sourceCode cell-code" id="cb71"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb71-1"><a href="#cb71-1" aria-hidden="true" tabindex="-1"></a>ggtree(tree_Acidobacteriota, layout<span class="op">=</span><span class="st">"circular"</span>)  <span class="op">%&lt;+%</span></span>
<span id="cb71-2"><a href="#cb71-2" aria-hidden="true" tabindex="-1"></a>  NEON_MAGs_metagenomes_chemistry <span class="op">+</span> </span>
<span id="cb71-3"><a href="#cb71-3" aria-hidden="true" tabindex="-1"></a>  geom_tiplab(size<span class="op">=</span><span class="dv">2</span>, hjust<span class="op">=-</span><span class="fl">.1</span>) <span class="op">+</span></span>
<span id="cb71-4"><a href="#cb71-4" aria-hidden="true" tabindex="-1"></a>  xlim(<span class="dv">0</span>,<span class="dv">28</span>) <span class="op">+</span></span>
<span id="cb71-5"><a href="#cb71-5" aria-hidden="true" tabindex="-1"></a>  geom_point(mapping<span class="op">=</span>aes(color<span class="op">=</span>Class, shape <span class="op">=</span> `Assembly Type`)) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Removed 305 rows containing missing values or values outside the scale range
(`geom_point()`).</code></pre>
</div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/tree1-output-2.png" id="tree1" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [49]:</pre></div><div id="cell-tree2" class="cell">
<div class="sourceCode cell-code" id="cb73"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb73-1"><a href="#cb73-1" aria-hidden="true" tabindex="-1"></a>ggtree(tree_bac, layout<span class="op">=</span><span class="st">"circular"</span>, branch.length<span class="op">=</span><span class="st">"none"</span>) <span class="op">+</span></span>
<span id="cb73-2"><a href="#cb73-2" aria-hidden="true" tabindex="-1"></a>    geom_hilight(node<span class="op">=</span><span class="dv">2673</span>, fill<span class="op">=</span><span class="st">"steelblue"</span>, alpha<span class="op">=</span><span class="fl">.6</span>) <span class="op">+</span></span>
<span id="cb73-3"><a href="#cb73-3" aria-hidden="true" tabindex="-1"></a>    geom_cladelab(node<span class="op">=</span><span class="dv">2673</span>, label<span class="op">=</span><span class="st">"Acidobacteriota"</span>, align<span class="op">=</span>TRUE, hjust<span class="op">=</span><span class="fl">1.1</span>,  </span>
<span id="cb73-4"><a href="#cb73-4" aria-hidden="true" tabindex="-1"></a>                  offset <span class="op">=</span> <span class="dv">0</span>, textcolor<span class="op">=</span><span class="st">'steelblue'</span>, barcolor<span class="op">=</span><span class="st">'steelblue'</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/tree2-output-1.png" id="tree2" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [50]:</pre></div><div id="a16780a3-08c3-45e8-a221-5e1449fddad9" class="cell">
<div class="sourceCode cell-code" id="cb74"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb74-1"><a href="#cb74-1" aria-hidden="true" tabindex="-1"></a>ggtree(tree_Acidobacteriota)  <span class="op">%&lt;+%</span></span>
<span id="cb74-2"><a href="#cb74-2" aria-hidden="true" tabindex="-1"></a>  NEON_MAGs_metagenomes_chemistry <span class="op">+</span> </span>
<span id="cb74-3"><a href="#cb74-3" aria-hidden="true" tabindex="-1"></a>  geom_tiplab(size<span class="op">=</span><span class="dv">2</span>, hjust<span class="op">=-</span><span class="fl">.1</span>) <span class="op">+</span></span>
<span id="cb74-4"><a href="#cb74-4" aria-hidden="true" tabindex="-1"></a>  xlim(<span class="dv">0</span>,<span class="dv">28</span>) <span class="op">+</span></span>
<span id="cb74-5"><a href="#cb74-5" aria-hidden="true" tabindex="-1"></a>  geom_point(mapping<span class="op">=</span>aes(color<span class="op">=</span>`soilInWaterpH`)) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/cell-50-output-1.png" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [51]:</pre></div><div id="cell-tree6" class="cell">
<div class="sourceCode cell-code" id="cb75"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb75-1"><a href="#cb75-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_metagenomes_chemistry_WREF <span class="op">&lt;-</span> NEON_MAGs_metagenomes_chemistry <span class="op">%&gt;%</span> </span>
<span id="cb75-2"><a href="#cb75-2" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(`Site ID.x` <span class="op">==</span> <span class="st">"WREF"</span>) <span class="op">%&gt;%</span> </span>
<span id="cb75-3"><a href="#cb75-3" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(Phylum <span class="op">==</span> <span class="st">"Acidobacteriota"</span>)</span>
<span id="cb75-4"><a href="#cb75-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb75-5"><a href="#cb75-5" aria-hidden="true" tabindex="-1"></a>WREF_MAGs_label <span class="op">&lt;-</span> NEON_MAGs_metagenomes_chemistry_WREF$label</span>
<span id="cb75-6"><a href="#cb75-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb75-7"><a href="#cb75-7" aria-hidden="true" tabindex="-1"></a>tree_bac_WREF_MAGs <span class="op">&lt;-</span>drop.tip(tree_bac,tree_bac$tip.label[<span class="op">-</span>match(WREF_MAGs_label, tree_bac$tip.label)])</span>
<span id="cb75-8"><a href="#cb75-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb75-9"><a href="#cb75-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb75-10"><a href="#cb75-10" aria-hidden="true" tabindex="-1"></a>ggtree(tree_bac_WREF_MAGs, layout<span class="op">=</span><span class="st">"circular"</span>)  <span class="op">%&lt;+%</span></span>
<span id="cb75-11"><a href="#cb75-11" aria-hidden="true" tabindex="-1"></a>  NEON_MAGs_metagenomes_chemistry <span class="op">+</span></span>
<span id="cb75-12"><a href="#cb75-12" aria-hidden="true" tabindex="-1"></a>  geom_point(mapping<span class="op">=</span>aes(color<span class="op">=</span>Phylum)) <span class="op">+</span></span>
<span id="cb75-13"><a href="#cb75-13" aria-hidden="true" tabindex="-1"></a>  theme(legend.title<span class="op">=</span>element_text(size<span class="op">=</span><span class="dv">5</span>), legend.text<span class="op">=</span>element_text(size<span class="op">=</span><span class="dv">5</span>), legend.spacing.y <span class="op">=</span> unit(<span class="fl">0.02</span>, <span class="st">"cm"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/tree6-output-1.png" id="tree6" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [52]:</pre></div><div id="cell-tree7" class="cell">
<div class="sourceCode cell-code" id="cb76"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb76-1"><a href="#cb76-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_metagenomes_chemistry_WREF <span class="op">&lt;-</span> NEON_MAGs_metagenomes_chemistry <span class="op">%&gt;%</span> </span>
<span id="cb76-2"><a href="#cb76-2" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(`Site ID.x` <span class="op">==</span> <span class="st">"WREF"</span>) </span>
<span id="cb76-3"><a href="#cb76-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb76-4"><a href="#cb76-4" aria-hidden="true" tabindex="-1"></a>WREF_MAGs_label <span class="op">&lt;-</span> NEON_MAGs_metagenomes_chemistry_WREF$label</span>
<span id="cb76-5"><a href="#cb76-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb76-6"><a href="#cb76-6" aria-hidden="true" tabindex="-1"></a>tree_bac_WREF_MAGs <span class="op">&lt;-</span>drop.tip(tree_bac,tree_bac$tip.label[<span class="op">-</span>match(WREF_MAGs_label, tree_bac$tip.label)])</span>
<span id="cb76-7"><a href="#cb76-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb76-8"><a href="#cb76-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb76-9"><a href="#cb76-9" aria-hidden="true" tabindex="-1"></a>ggtree(tree_bac_WREF_MAGs, layout<span class="op">=</span><span class="st">"circular"</span>)  <span class="op">%&lt;+%</span></span>
<span id="cb76-10"><a href="#cb76-10" aria-hidden="true" tabindex="-1"></a>  NEON_MAGs_metagenomes_chemistry <span class="op">+</span></span>
<span id="cb76-11"><a href="#cb76-11" aria-hidden="true" tabindex="-1"></a>  geom_point(mapping<span class="op">=</span>aes(color<span class="op">=</span>Phylum)) <span class="op">+</span></span>
<span id="cb76-12"><a href="#cb76-12" aria-hidden="true" tabindex="-1"></a>  theme(legend.title<span class="op">=</span>element_text(size<span class="op">=</span><span class="dv">5</span>), legend.text<span class="op">=</span>element_text(size<span class="op">=</span><span class="dv">5</span>), legend.spacing.y <span class="op">=</span> unit(<span class="fl">0.02</span>, <span class="st">"cm"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/tree7-output-1.png" id="tree7" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [53]:</pre></div><div id="ffad69c6-3e40-4b44-a295-39c7851c49e6" class="cell">
<div class="sourceCode cell-code" id="cb77"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb77-1"><a href="#cb77-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_metagenomes_chemistry_soilInCaClpH <span class="op">&lt;-</span> NEON_MAGs_metagenomes_chemistry <span class="op">%&gt;%</span> </span>
<span id="cb77-2"><a href="#cb77-2" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(soilInCaClpH <span class="op">&lt;</span> <span class="dv">0</span>) <span class="op">%&gt;%</span> </span>
<span id="cb77-3"><a href="#cb77-3" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(Domain <span class="op">==</span> <span class="st">"Bacteria"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [54]:</pre></div><div id="caf0188f-2d72-4a30-97ff-b75ededfed25" class="cell">
<div class="sourceCode cell-code" id="cb78"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb78-1"><a href="#cb78-1" aria-hidden="true" tabindex="-1"></a>soilInCaClpH_MAGs_label <span class="op">&lt;-</span> NEON_MAGs_metagenomes_chemistry_soilInCaClpH</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [55]:</pre></div><div id="787c6d4e-9525-4265-a4bf-e6ea170f6a67" class="cell">
<div class="sourceCode cell-code" id="cb79"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb79-1"><a href="#cb79-1" aria-hidden="true" tabindex="-1"></a>tree_bac_soilInCaClpH_MAGs <span class="op">&lt;-</span>drop.tip(tree_bac,tree_bac$tip.label[<span class="op">-</span>match(soilInCaClpH_MAGs_label, tree_bac$tip.label)])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [56]:</pre></div><div id="9cc58978-175c-4a18-b651-60f36e66dfd8" class="cell">
<div class="sourceCode cell-code" id="cb80"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb80-1"><a href="#cb80-1" aria-hidden="true" tabindex="-1"></a>ggtree(tree_bac_soilInCaClpH_MAGs, layout<span class="op">=</span><span class="st">"circular"</span>)  <span class="op">%&lt;+%</span></span>
<span id="cb80-2"><a href="#cb80-2" aria-hidden="true" tabindex="-1"></a>  NEON_MAGs_metagenomes_chemistry <span class="op">+</span></span>
<span id="cb80-3"><a href="#cb80-3" aria-hidden="true" tabindex="-1"></a>  geom_point(mapping<span class="op">=</span>aes(color<span class="op">=</span>Phylum)) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/cell-56-output-1.png" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [57]:</pre></div><div id="cell-tree3" class="cell">
<div class="sourceCode cell-code" id="cb81"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb81-1"><a href="#cb81-1" aria-hidden="true" tabindex="-1"></a>NEON_MAGs_metagenomes_chemistry_soilInCaClpH <span class="op">&lt;-</span> NEON_MAGs_metagenomes_chemistry <span class="op">%&gt;%</span> </span>
<span id="cb81-2"><a href="#cb81-2" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(soilInWaterpH <span class="op">&lt;</span> <span class="dv">5</span>) <span class="op">%&gt;%</span> </span>
<span id="cb81-3"><a href="#cb81-3" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(Domain <span class="op">==</span> <span class="st">"Bacteria"</span>)</span>
<span id="cb81-4"><a href="#cb81-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Make a vector of the MAGs labels</span></span>
<span id="cb81-5"><a href="#cb81-5" aria-hidden="true" tabindex="-1"></a>soilInCaClpH_MAGs_label <span class="op">&lt;-</span> NEON_MAGs_metagenomes_chemistry_soilInCaClpH$label</span>
<span id="cb81-6"><a href="#cb81-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb81-7"><a href="#cb81-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Use appropriate Bacteria or Archaea tree to select your site MAG labels</span></span>
<span id="cb81-8"><a href="#cb81-8" aria-hidden="true" tabindex="-1"></a>tree_bac_soilInCaClpH_MAGs <span class="op">&lt;-</span>drop.tip(tree_bac,tree_bac$tip.label[<span class="op">-</span>match(soilInCaClpH_MAGs_label, tree_bac$tip.label)])</span>
<span id="cb81-9"><a href="#cb81-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb81-10"><a href="#cb81-10" aria-hidden="true" tabindex="-1"></a><span class="co"># Now make a tree with your MAGs of interest</span></span>
<span id="cb81-11"><a href="#cb81-11" aria-hidden="true" tabindex="-1"></a>ggtree(tree_bac_soilInCaClpH_MAGs, layout<span class="op">=</span><span class="st">"circular"</span>)  <span class="op">%&lt;+%</span></span>
<span id="cb81-12"><a href="#cb81-12" aria-hidden="true" tabindex="-1"></a>  NEON_MAGs_metagenomes_chemistry <span class="op">+</span></span>
<span id="cb81-13"><a href="#cb81-13" aria-hidden="true" tabindex="-1"></a>  geom_point(mapping<span class="op">=</span>aes(color<span class="op">=</span>Phylum)) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/tree3-output-1.png" id="tree3" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [58]:</pre></div><div id="8d386c25-fa45-44e6-8336-11932b561258" class="cell">
<div class="sourceCode cell-code" id="cb82"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb82-1"><a href="#cb82-1" aria-hidden="true" tabindex="-1"></a>NEON_chemistry_WREF <span class="op">&lt;-</span> NEON_chemistry <span class="op">%&gt;%</span></span>
<span id="cb82-2"><a href="#cb82-2" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(siteID <span class="op">==</span> <span class="st">'WREF'</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [59]:</pre></div><div id="cell-plot13" class="cell">
<div class="sourceCode cell-code" id="cb83"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb83-1"><a href="#cb83-1" aria-hidden="true" tabindex="-1"></a>  NEON_chemistry_WREF <span class="op">%&gt;%</span></span>
<span id="cb83-2"><a href="#cb83-2" aria-hidden="true" tabindex="-1"></a>  ggplot(aes(x <span class="op">=</span> soilInWaterpH, y<span class="op">=</span> soilTemp)) <span class="op">+</span></span>
<span id="cb83-3"><a href="#cb83-3" aria-hidden="true" tabindex="-1"></a>  geom_point(aes(color<span class="op">=</span> nlcdClass)) <span class="op">+</span></span>
<span id="cb83-4"><a href="#cb83-4" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">45</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>), axis.text.y <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">5</span>)) <span class="op">+</span> </span>
<span id="cb83-5"><a href="#cb83-5" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Soil pH v Soil Temp at WREF"</span>, x <span class="op">=</span> <span class="st">"Soil in Water pH"</span>, y <span class="op">=</span> <span class="st">"Soil Temp"</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot13-output-1.png" id="plot13" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [60]:</pre></div><div id="cell-plot14" class="cell">
<div class="sourceCode cell-code" id="cb84"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb84-1"><a href="#cb84-1" aria-hidden="true" tabindex="-1"></a>NEON_chemistry_WREF <span class="op">%&gt;%</span>   </span>
<span id="cb84-2"><a href="#cb84-2" aria-hidden="true" tabindex="-1"></a>ggplot(aes(x <span class="op">=</span> `elevation`, y <span class="op">=</span> `genomicsSampleID`, color <span class="op">=</span> `soilTemp`)) <span class="op">+</span></span>
<span id="cb84-3"><a href="#cb84-3" aria-hidden="true" tabindex="-1"></a>  geom_point() <span class="op">+</span></span>
<span id="cb84-4"><a href="#cb84-4" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Elevation of Samples at WREF"</span>, x <span class="op">=</span> <span class="st">"Elevation"</span>, y <span class="op">=</span> <span class="st">"Sample ID"</span>) <span class="op">+</span></span>
<span id="cb84-5"><a href="#cb84-5" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">45</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot14-output-1.png" id="plot14" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [61]:</pre></div><div id="cell-plot15" class="cell">
<div class="sourceCode cell-code" id="cb85"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb85-1"><a href="#cb85-1" aria-hidden="true" tabindex="-1"></a>Acido_Data <span class="op">%&gt;%</span> </span>
<span id="cb85-2"><a href="#cb85-2" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(`Site ID`<span class="op">==</span> <span class="st">'WREF'</span>) <span class="op">%&gt;%</span></span>
<span id="cb85-3"><a href="#cb85-3" aria-hidden="true" tabindex="-1"></a>  ggplot(aes(x <span class="op">=</span> fct_rev(fct_infreq(Class)), fill <span class="op">=</span> Order)) <span class="op">+</span></span>
<span id="cb85-4"><a href="#cb85-4" aria-hidden="true" tabindex="-1"></a>  geom_bar() <span class="op">+</span></span>
<span id="cb85-5"><a href="#cb85-5" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">45</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>), axis.text.y <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">5</span>)) <span class="op">+</span>  </span>
<span id="cb85-6"><a href="#cb85-6" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Acidobacteria Class Count in WREF"</span>, x <span class="op">=</span> <span class="st">"Class"</span>, y <span class="op">=</span> <span class="st">"Count"</span>) <span class="op">+</span></span>
<span id="cb85-7"><a href="#cb85-7" aria-hidden="true" tabindex="-1"></a>  theme_classic()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot15-output-1.png" id="plot15" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>
<div class="cell-container"><div class="cell-decorator"><pre>In [62]:</pre></div><div id="cell-plot16" class="cell">
<div class="sourceCode cell-code" id="cb86"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb86-1"><a href="#cb86-1" aria-hidden="true" tabindex="-1"></a>Acido_Data <span class="op">%&gt;%</span> </span>
<span id="cb86-2"><a href="#cb86-2" aria-hidden="true" tabindex="-1"></a>  <span class="bu">filter</span>(`Site ID`<span class="op">==</span> <span class="st">'WREF'</span>) <span class="op">%&gt;%</span></span>
<span id="cb86-3"><a href="#cb86-3" aria-hidden="true" tabindex="-1"></a>  ggplot(aes(x <span class="op">=</span> fct_rev(fct_infreq(Class)), y<span class="op">=</span> `Total Number of Bases`, fill <span class="op">=</span> Order)) <span class="op">+</span></span>
<span id="cb86-4"><a href="#cb86-4" aria-hidden="true" tabindex="-1"></a>  geom_point(aes(color<span class="op">=</span> Order)) <span class="op">+</span></span>
<span id="cb86-5"><a href="#cb86-5" aria-hidden="true" tabindex="-1"></a>  theme(axis.text.x <span class="op">=</span> element_text(angle<span class="op">=</span><span class="dv">45</span>, vjust<span class="op">=</span><span class="dv">1</span>, hjust<span class="op">=</span><span class="dv">1</span>), axis.text.y <span class="op">=</span> element_text(size <span class="op">=</span> <span class="dv">5</span>)) <span class="op">+</span>  </span>
<span id="cb86-6"><a href="#cb86-6" aria-hidden="true" tabindex="-1"></a>  labs(title <span class="op">=</span> <span class="st">"Total Number of Bases in Acido Classes in WREF"</span>, x <span class="op">=</span> <span class="st">"Class"</span>, y <span class="op">=</span> <span class="st">"Number of Bases"</span>) <span class="op">+</span></span>
<span id="cb86-7"><a href="#cb86-7" aria-hidden="true" tabindex="-1"></a>  theme_classic()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-display">
<div class="">
<figure class="figure">
<p class=""><img src="code.embed_files/figure-html/plot16-output-1.png" id="plot16" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div></div>


     </main> <!-- /main -->  <script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>  </div> <!-- /content --> 
  
</body></html>